# Measurement Process  
Overview of the 16S rRNA marker gene metagenomics measurement process including the sources of error and bias introduced at different step in the process along with different mitigation strategies that have been developed to address each of the issues.    

Diagram of measurement process \@ref(fig:fishbone) .  
```{r fishbone, echo=FALSE, fig.cap="Fishbone diagram of the 16S rRNA metagenomic sequencing measurement process."}
library(qcc)
cause.and.effect(
  cause=list("Sample Collection"=c("Sampling","Storage Conditions"),
             "DNA Extraction"=c("Cell Lysis", "Purification"),
             "PCR"=c("Primers", "Variable Region",
                     "Polymerase","Cycling\n Condiitons"),
             "Library Prep"=c("Normalization", "Pooling"),
             "Sequencing"=c("Chemistry", "Monitoring", 
                            "Base calling"),
             "Sequence Processing"=c("Read Filtering", "Chimera Filtering",
                                     "OTU Assignment")),
                 effect="OTU\nTable",
  title = "16S rRNA Sequencing Measurement Process")
```


Review of what is currently known about the sources of error and bias in the 16S metagenomics measurement process. 
Cause-effect diagrams are commonly used in process quality control to show the relationship between steps in the measurement process and sources of error or bias \@ref(fig:cause-effect).  
Potentially may want to revise to focus on sources of error, e.g. cell type bias, preferential amplification, sequence errors, chimeras .

```{r cause-effect, echo=FALSE, fig.cap = "Causes (branch labels) of bias and variability indicated for different sources of error (branch text)."}
cell_bias <- c("Storage\nConditions","DNA extraction")
amp_bias <- c("PCR Primers", "Cycling Conditions")
seq_error <- c("Polymerase","Seq Rxn","Rxn Monitoring","Base Calling")
otu_inflation <- c("Variable Region", "Diversity\ninhomogeneity","Seq Error","Chimera")
coverage <- c("Normalization", "DNA Quant","PCR inhibitors")

cause.and.effect(
  cause=list("Cell Type"=cell_bias, "Amplification"=amp_bias,"OTU Inflation"= otu_inflation, "Uneven Sampling"=coverage),
                 effect="OTU\nTable",
  title = "16S rRNA Sequencing Cause-Effect")

```

## Sample to Seq 
The first part of the measurement process is generating sequencing data from an environmental sample.  

The part consists of five main steps:  

1. Sample Collection  
1. Sample Processing  
1. 16S rRNA PCR  
1. Library Prep   
1. Sequencing  

### Sample Collection
* Collection of sample that is representative of population being studied.  
  * Kitty Biome study as an example of sample heterogeneity.  
* Impact of sample storage.  
  * Fecal storage study.  
  * Old coral study?  

### Sample Processing
* DNA extraction  
  * Cell type specific extraction biases  
  * DNA concentration requirements for library prep  
  * Issues with low concentration samples - skin microbiome  
* Additional DNA purification to remove PCR inhibitors.  
  * Old PCR inhibitor paper.  
  * Any studies showing inhibitors inducing biases in metagenomic studies?  
    * Potentially sample specific biases but do not know of any within sample biases  

### 16S rRNA PCR
* Template concentration  
  * This is related to low concentration samples or samples with poor DNA extraction efficiencies.
  * @Kennedy2014 showed that the profile of replicate samples for different template concentration was more variable then when the imput concentration is the same. Pooling replicate PCR reactions decreased the within sample variablility. 
* Target variable region  
* PCR primers (optional barcoded primers) 
* PCR reagents  
  * Polymerase - high fidelity  
  * Reagent contaminats  
* PCR cycling conditions  
  * number of cycles
  * sequence copy abundance  
  * chimeras  

### Library Preparation  
@Kennedy2014 found that variability due to PCR (particularly template concentration) resulted in less variability for within sample results than sample preparation (library preparation) and inter-lane variability. 
Barcoded 16S PCR primers were used in the study therefore the impact of barcodes and variability of 16S PCR was confounded.  

* Barcode/ indexing (optional)  
  * Do not know of any barcode specific biases
* Normalization and Pooling  
  * Methods
    * DNA quantification  
    * Bead based assays
  * Associated Biases  
    * Uneven sequencing depth - impact of coverage  

### Sequencing  
* Sequencing platform  
* Read length  
* Sequencing chemistry  

## Seq to OTU  
Convering raw sequence data to a OTU table with taxonomic annotations for use in downstream applications.

### quality filter/ read trimming  
* Filtering low quality reads - short reads with high error rates.  
* Trimming reads - removing ends of reads based on base quality score as well as removal of adapter sequences that were added during sample processing.
* Merging paired ends
  - Merging forward and reverse reads into single contigs
* chimera detection
  - detection and removal of chimeric reads, artifact of the PCR amplification process.  

### OTU Assignment  
OTU assignment most commonly by clustering is used to reduce the number of spurious observations, sequences are grouped into OTUs.  

* Different approaches to clustering  
    + distance based clustering - most commonly used  
      * Distance based clustering is a NP hard problem (i.e. computationaly intense), therefore a number of heuristics are used to make the method computationally tractable for large datasets.
      * Threshold are used to define the similarity of sequences within a cluster, 99% and 97% are commonly used. 
        - Some biological and sequencing error basis for threshold values, (e.g. 99% species level, 97% genus level, 1% error rate). Taxonomic variability in 16S rRNA sequence diversity (Pai?) and sequencing error [@Huse2010, @Schirmer2015, @Amore2016] challenges the validity of these values.
      * Different distance based thresholds.
        - average linkage  
        - single linkage  
        - complete linkage  
    + Other clustering/ OTU assignment methods include:
      * distribution based  
      * oligotyping  
      * tree based clustering [@Wu2013]
    + De novo vs Reference based clustering
      * Sequences can be clustered with a set of reference cluster centers or without (_de novo_).  

### Taxonomic Assignment
Assign clusters a taxonomic name  

## Downstream analysis  
### Differential Abundance  
* Differences in taxa or OTU abundance between experimental factors, e.g. treatment vs. control.  

### Alpha Diversity  
* Measure of biological or ecological diversity within a sample.  
* Alpha diversity as a continuum related to the metric dependence on richness and evenness [@Chao2014]
* Four primary categories richness, evenness, abundance weighted, and dissimilarity [@Ricotta2007]
  * Richness
    * Hill numbers, rarefaction, and richness estimation [@Chao2014]
  * Evenness
    * Criteria for evaluating evenness metrics [@Tuomisto2012]
    * See @Tuomisto2012 Table 1 for summary of evenness metric including interpretation
  * Abundance weighted
  * Dissimilarity
* Parametric vs. Non-parametric diversity metrics for 16S rRNA sequence analysis [@Bunge2014]
* Development of alpha diversity richness metric for microbial diversity.  
  * Chao1 a traditional diversity metric estimates total diversity using the number of singletons, OTUs representing only a single sequence.  
    * Issue with singleton based diversity metrics for microbial diversity is that for incresed sequencing depth the number of singletons with continue to increase due to sequencing errors, leading to artifically inflated diversity estimates. 
  * @Chiu2015 developed a modified version of the Chao1 estimator that is more robust to singletons bue to sequencing errors. 
  * This method estimates diversity based on the number of doubletons to tentons.  
  * Incorporates Hill numbers as well.  

### Beta Diversity  

