# Measurement Process  
Overview of the 16S rRNA marker gene metagenomics measurement process including the sources of error and bias introduced at different step in the process along with different mitigation strategies that have been developed to address each of the issues. 
See @Goodrich2014 for a review of 16S rRNA marker gene metagenomics measurement process and general recomendations for conducting a microbiome study.  

@Zhou2011 Used biological and technical replicates with a _Shewanella oneidenis MR-1_ to assess the reproducibility and quanitative accuracy of 16S sequencing, using 454 pyroseuqencing and targeting the V45 region. Results demonstrated a lack of reporducibility and quantiative accuracy when comparing the $\beta$ diversity and relative abundance between replicates. 

Diagram of measurement process \@ref(fig:fishbone).  
```{r fishbone, echo=FALSE, fig.cap="Fishbone diagram of the 16S rRNA metagenomic sequencing measurement process."}
library(qcc)
cause.and.effect(
  cause=list("Sample Collection"=c("Sampling","Storage Conditions"),
             "DNA Extraction"=c("Cell Lysis", "Purification"),
             "PCR"=c("Primers", "Variable Region",
                     "Polymerase","Cycling\n Condiitons"),
             "Library Prep"=c("Normalization", "Pooling"),
             "Sequencing"=c("Chemistry", "Monitoring", 
                            "Base calling"),
             "Sequence Processing"=c("Read Filtering", "Chimera Filtering",
                                     "OTU Assignment")),
                 effect="OTU\nTable",
  title = "16S rRNA Sequencing Measurement Process")
```


Review of what is currently known about the sources of error and bias in the 16S metagenomics measurement process. 
Cause-effect diagrams are commonly used in process quality control to show the relationship between steps in the measurement process and sources of error or bias \@ref(fig:cause-effect).  
Potentially may want to revise to focus on sources of error, e.g. cell type bias, preferential amplification, sequence errors, chimeras .

```{r cause-effect, echo=FALSE, fig.cap = "Causes (branch labels) of bias and variability indicated for different sources of error (branch text)."}
cell_bias <- c("Storage\nConditions","DNA extraction")
amp_bias <- c("PCR Primers", "Cycling Conditions")
seq_error <- c("Polymerase","Seq Rxn","Rxn Monitoring","Base Calling")
otu_inflation <- c("Variable Region", "Diversity\ninhomogeneity","Seq Error","Chimera")
coverage <- c("Normalization", "DNA Quant","PCR inhibitors")

cause.and.effect(
  cause=list("Cell Type"=cell_bias, "Amplification"=amp_bias,"OTU Inflation"= otu_inflation, "Uneven Sampling"=coverage),
                 effect="OTU\nTable",
  title = "16S rRNA Sequencing Cause-Effect")

```

## Sample to Seq 
The first part of the measurement process is generating sequencing data from an environmental sample.  

The part consists of five main steps:  

1. Sample Collection  
1. Sample Processing  
1. 16S rRNA PCR  
1. Library Prep   
1. Sequencing  


### Sample Collection  

- Collection of sample that is representative of population being studied  
    - Kitty Biome study as an example of sample heterogeneity  
- Impact of sample storage   
    - @Flores2015 evaluated impact of storage conditions on 16S metagenomics.  The study results supported the use of RNALater as storage media after evaluating the different temperature, media, and duration for biological replicate stool samples. The results were evaluated using intraclass correlation coefficients as well as $\alpha$ and $\beta$ diversity. __DATA NOT PUBLICALLY AVAILABLE!!!!!__    
    - Old coral study?  
    - @Hill2016 evlauated the use of DNA Genotek storage vial on microbiomes for infant and elderly samples. 
    Found OTU specific differential abundance results. 
    Elderly samples were less impacted by storage than infant samples.     Results suggest that the impact of storage conditions is organism specific. 
    - @Lauber2011 study evaluating the impact of storage conditions, weighted UniFrac community analysis not impacted by storage conditions, whereaas for skin communities storage conditions impacted  unweighted UniFrac results.  
    - @Wu2010 another study evaluating the impact of storage conditions  
    - @Sinha2016 compared impact of storage and sample collection methods for fecal microbiome studies as part of an interlaboratory study. 
    Found storage methods produced similar results to samples immediately processed.
    Differences in results between laboratories are likely due to differences in sample processing methods (e.g. DNA extraction method, PCR primers, target region) than a laboratory effect.  

### Sample Processing

#### DNA extraction 

Considerations:  

* Cell type specific extraction biases   
* DNA concentration requirements for library prep   
    * Issues with low concentration samples - skin microbiome   
* Extra cellular DNA  
    * Nucleic acids exuded from cells  
    * Relic DNA, DNA from dead cells.  
        * @Carini2016 used Propidium monoazide (PMA) to evaluate the impact of DNA from dead cells on 16S and ITS metagenomic diversity estimates. For both the fungal and microbial communities 40% of the DNA was from dead cells as a result a significant proportion of the oberved microbial diversity was from dead cells.  
        * Need to take the presence of relic DNA into consideration when interpreting metagenomic results, not just characterizing the active, viable, or living cells at point of sampling, but also system history.  
        * Sequencing rRNA instead of rDNA is also used to characterize the active component of the microbial community [@Blazewicz2013]. Though this method it's own limiations.
        * See @Cangelosi2014 for other methods for targeting viable microbial community.
        
* PCR inhibitiors   
    * Old PCR inhibitor paper.   
    * Any studies showing inhibitors inducing biases in metagenomic studies?    
        * Potentially sample specific biases but do not know of any within sample biases  

* Cell type extraction biases   
    * @Albertsen2015 evalauted the impact of different DNA extraction methods for activated slude communities.  Bead beating both time and intensity impacted the yield and size distribution of the extracted DNA as well as the observed abundance of different bacteria at both the phylum level and class for Proteobacteria.  
    * Extraction methods vary in there ability to extract DNA from hard to lyse cells [@Olson2012], e.g. Gram positives, and reproducibiliy of the results [@Carrigg2007] [@Ettenauer2012].  
    * No known method to correct for or quantify extraction biases.  
    * Whole cells control materials may provide a way to correct for cell type specific extraction biases.  
    * Limitation to comparing results from multiple studies when different extraction methods were used.  


### 16S rRNA PCR  

#### Template concentration  

* This is related to low concentration samples or samples with poor DNA extraction efficiencies.   

@Kennedy2014 showed that the profile of replicate samples for different template concentration was more variable then when the imput concentration is the same. Pooling replicate PCR reactions decreased the within sample variablility.    

@Amore2016 increasing the template concentration from 1 ng to 10 ng for MiSeq lead to higher percentages of chimeric reads 0.08 compared to 0.2% (p = 0.20).   

#### Target variable region    
@Clooney2016 Compared sequencing data between platforms and variable regions. Based on the relative abundance of genera present in 20% of the datasets (including whole metagenome sequencing (WMS)) the V12 and V45 primer regions, PGM sequencing data was less impacted than MiSeq to different variable regions.   

@Kozich2013 compared the results for three different variable regions sequenced using the MiSeq platform. The three regions, V34, V4, and V45 represented different length (250 bp, 430 bp, and 375 bp) though overlapping target regions. The observed error rate varied between run as well as variable regions resulting in differences in predicted OTUs. V45 had the highest predicted OTUs for all samples and replicate sequencing runs, followed by V34 and V4. The number of observed OTUs was higher for all datasets than expected for the mock community datasets.   

#### PCR primers (optional barcoded primers)  

@Albertsen2015 performed both an _in silico_ and _in vitro_ comparison of primer regions as well as comparing the relative abundance of different Phyla to metagenomic and metatranscriptomic datasets. The author's concluded by recomending V13 for activated sludge samples based on what is known about the community composition of the system.   

@Esling2015 developed a latin square barcode design to facilitate detection and removal of cross-contamination along with tools for implenting their recomended barcoding methods, https://github.com/esling/mistagging.  

@Amore2016 evaluated the impact of primer mismatches using a mock community dataset by correlating abundance with mismatches.  

@Engelbrektson2010 explored the impact of different primers, variable regions, and amplicon legnths on $\alpha diversity metrics for termine gut samples. Amplicon lengths impacted diversity and richness estimates but primers targeting the same region had consistent results.   

#### PCR reagents  
* Error rates vary based on the polymerase being used, it is commonly recommended that a high fidelity polymerase is used.
* DNA polymerase from organisms can remain in extracted DNA after purification and potentially amplify the organisms DNA and potentially risk skewing experiments when comparing microbiomes from different environments [@Marx2016]. 
This has not been tested therefore the impact is unknown.   
* Can calcualte the errors due to PCR using first principles [@Marx2016]
* Polymerase - high fidelity  
* Reagent contaminats  

#### PCR cycling conditions    
#### number of cycles  

Using a mock community of 68 organisms @Amore2016 showed that increasing the number of PCR cycles resulted in increased sequencing error rates (0.58% vs. 0.64% p = 0.11) and chimeras (15 cycles 0.00% and 25 cycles 0.66%, p = 0.0245).  

@Albertsen2015 evaluated the amount of template and cycle number but only observed "small changes". Decreasing the annealing temperatures of 58C and 52C resulted in increased abundance of prevously low abundance OTU with no impact on high abundance OTUs.  

@Yu2015 compared standard and nest PCR (barcodes used in the second round), showed that the number of cycles and use of separate PCR and indexing reactions influenced the number of observed OTUs and $\beta$ diversity.   

#### Sequence Copy Abundance  

#### Chimeras  

### Library Preparation  

@Kennedy2014 found that variability due to PCR (particularly template concentration) resulted in less variability for within sample results than sample preparation (library preparation) and inter-lane variability. 
Barcoded 16S PCR primers were used in the study therefore the impact of barcodes and variability of 16S PCR was confounded.   

@Schirmer2015 sequenced a mock community of 59 different strains to evaluate the sequencing error rate for different sample processing methods. Though only fully characterizing a subset of a the results.  

#### Barcode/ indexing (optional)  

* Do not know of any barcode specific biases  

#### Normalization and Pooling  

* Methods  
    * DNA quantification  
    * Bead based assays  
* Associated Biases   
    * Uneven sequencing depth - impact of coverage  

### Sequencing  
* Sequencing platforms
    * Illumina - MiSeq, HiSeq, GAII 
    * Ion Torrent  
    * PacBio  
    * Oxford Nanopore - Minion  
* Factors impacting measurement results  
    * Read length    
    * Sequencing chemistry  
* Studies comparing platforms  
    * @Clooney2016 compared 16S rRNA sequence data for environmental samples sequenced using two platforms MiSeq and PGM. Comparing results between stool samples from six individuals based on the relative abundance of genera present in 20% of the datasets (including whole metagenome sequencing (WMS)) the datasets clustered together based on sequencing methods (platform and primers) rather than individual, with WMS clustering seperately form the 16S results [@Clooney2016, figure 1].  

* Sequencing errors  
    * Random errors - general error rates  
    * Systematic sequencing errors  
    * How systematic and random errors impact diversity studies.

## Seq to OTU 

Convering raw sequence data to a OTU table with taxonomic annotations for use in downstream applications.

### Preprocessing 
* Filtering and Trimming
    - Filtering low quality reads - short reads with high error rates.  
    - Trimming reads - removing ends of reads based on base quality score as well as removal of adapter sequences that were added during sample processing.  
    - Approaches
        + Quality Score
            * min quality score
                - Illumina QS2  
            * Window average  
        + Length  
            * Min length  
        + Trimming left and right ends  
        + Barcode and primer based  
    - Tools  
        + Trimmomatic  
        + Sickle  
        + fastx  
* Error correction  
    - BayesHammer  
    - DADA2  
    - Deblur?  
* Merging paired ends
  - Merging forward and reverse reads into single contigs  
      + Methods
          * Perfect overlap - DADA2 
          * dynamic programming - mothur  
          * Probabilistic/ Optimization   
              - FLASH  
              - PANDAseq  @masella2012pandaseq
                + Uses a probabilitistic approach to generating contigs from paired-end reads 
                    * First finds primers in forward and reverse sequences (optional)  
                    * Determines overlap by maximizing the Probablity of the contig for overlap length greater than the user defined value.
                        - Bases in Non-overlaping regions assigned probability of 1/4  
                        - Bases in overlaping regions assigned a probability based on the assigned base quality score and whether the bases agree between the forward and reverse reads.  
                        _ Additional algorithms available; ea utils fastqJoin, FLASH, PEAR, RDP, STICH, and UPARSE 
                        
                    * Assigns base and quality score for overlap regions  
                        - for disagreements assigns base for higher quality score and assigns probability using a probabilistic method using the fastq quality scores
                    * Calculate contig average error - geometric mean [$\left(\prod^n_{i=1}x_i\right)^{1/n}$]of the assigned quality scores
                + Calculates the probability of an observed base using the assigned quality scores  
                    * conditional probability that the true nucleotides for the aligned paired end reads given the observed nucleotides  
                    * Assumes equal probablity for alternative (non-observed bases)  
                    * Option to use primer sequences location and removal during overlap generation  
                    
                + Filter/ validating contigs 
                    * average contig quality score (removed in v2.8 due to bug) 
                    * contig length   
                    * ambiguous bases  
                    * optional - known secondary structure, conserved regions, not implemented in distribution
                    * statistical methods - pear false positive test  
                + Method evaluation  
                    * Simulated data - test algorithm correctness  
                    * single template amplicon PCR - check contig quality  
                    * defined fragment mixture to check assembly yields  
                    
              - PEAR
                + Algorithm Summary  
                    * Optimizes assembly score for based on base quality score and mismatches 
                    * See @zhang2014pear for alignment base probability calculations  
                    * Developed Observed Expected alignment score (OES) for use in testing significance of merge 
                + Method Evaluation
                    * simulated data comparison of multiple paired-end assembly methods for multiple fragment sizes and overlap 
                    * Compared based on the number of merged contigs and false positive rate, also assessed computational cost  
                    * Also evaluated using single organism datasets
                    
              
* chimera detection
  - detection and removal of chimeric reads, artifact of the PCR amplification process.  
      + ChimeraSlayer  
      + uchime  
      + DADA2 - bimera  

### OTU Assignment  

OTU assignment most commonly by clustering is used to reduce the number of spurious observations, sequences are grouped into OTUs.  

* Different approaches to clustering  
    + distance based clustering - most commonly used  
        * Distance based clustering is a NP hard problem (i.e. computationaly intense), therefore a number of heuristics are used to make the method computationally tractable for large datasets.
        * Threshold are used to define the similarity of sequences within a cluster, 99% and 97% are commonly used. 
            - Some biological and sequencing error basis for threshold values, (e.g. 99% species level, 97% genus level, 1% error rate). Taxonomic variability in 16S rRNA sequence diversity (Pai?) and sequencing error [@Huse2010, @Schirmer2015, @Amore2016] challenges the validity of these values.
        * Different distance based thresholds.
            - average linkage  
            - single linkage  
            - complete linkage  
    + Other clustering/ OTU assignment methods include:
        * distribution based  
        * oligotyping  
        * tree based clustering [@Wu2013]
    + De novo vs Reference based clustering
        * Sequences can be clustered with a set of reference cluster centers or without (_de novo_).  

### Taxonomic Assignment

Assign clusters a taxonomic name
* @Claesson2010 _in silico_ analysis of the taxonomic resolution for different variable regions and read lengths (454 and MISeq 150, 100, 75, and 50 bp paired end reads) and primers.

## Downstream analysis  
### Differential Abundance  

* Differences in taxa or OTU abundance between experimental factors, e.g. treatment vs. control.  
* Similarity to RNA-seq/ microarray differential abundance   
* Key factors  
    - Effect size, sample size, abundance  
* See @Jonsson2016 for summary of different methods  
* Overview of different methods and approaches  
    - Statistical distributions   
    - Normalization methods   
    - Transformations   

### Alpha Diversity  

* Measure of biological or ecological diversity within a sample.  
* Alpha diversity as a continuum related to the metric dependence on richness and evenness [@Chao2014]
* Four primary categories richness, evenness, abundance weighted, and dissimilarity [@Ricotta2007]
    * Richness
        * Hill numbers, rarefaction, and richness estimation [@Chao2014]
    * Evenness
        * Criteria for evaluating evenness metrics [@Tuomisto2012]
        * See @Tuomisto2012 Table 1 for summary of evenness metric including interpretation
    * Abundance weighted
    * Dissimilarity
* Parametric vs. Non-parametric diversity metrics for 16S rRNA sequence analysis [@Bunge2014]
* Development of alpha diversity richness metric for microbial diversity.  
    * Chao1 a traditional diversity metric estimates total diversity using the number of singletons, OTUs representing only a single sequence.  
        * Issue with singleton based diversity metrics for microbial diversity is that for incresed sequencing depth the number of singletons with continue to increase due to sequencing errors, leading to artifically inflated diversity estimates. 
    * @Chiu2015 developed a modified version of the Chao1 estimator that is more robust to singletons bue to sequencing errors. 
    * This method estimates diversity based on the number of doubletons to tentons.  
    * Incorporates Hill numbers as well.  

### Beta Diversity  

## Step-by-Step protocols  

* @Albertsen2015 Activated Sludge http://www.midasfieldguide.org/
* List of tools for each step in the sequence analysis workflow http://h3abionet.org/tools-and-resources/sops/16s-rrna-diversity-analysis 
