<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Measurement Science of Metagenomics</title>
  <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
  <meta name="description" content="Measurement Science of Metagenomics">
  <meta name="generator" content="bookdown <!--bookdown:version--> and GitBook 2.6.7">




  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<!--bookdown:link_prev-->
<!--bookdown:link_next-->

<link href="libs/gitbook/css/style.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-fontsettings.css" rel="stylesheet" />
<script src="libs/gitbook/js/app.js"></script>
<script src="libs/gitbook/js/lunr.js"></script>
<script src="libs/gitbook/js/plugin-search.js"></script>
<script src="libs/gitbook/js/plugin-sharing.js"></script>
<script src="libs/gitbook/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook/js/plugin-bookdown.js"></script>
<script src="libs/gitbook/js/jquery.highlight.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>


<!--bookdown:title:start-->
<div id="header">
<h1 class="title">Measurement Science of Metagenomics</h1>
</div>
<!--bookdown:title:end-->

<!--bookdown:toc:start-->
  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">
<!--bookdown:toc2:start-->
<ul>
<li><a href="#metrology-of-metagenomics"><span class="toc-section-number">1</span> Metrology of Metagenomics</a><ul>
<li><a href="#microbiome"><span class="toc-section-number">1.1</span> Microbiome</a></li>
<li><a href="#measuring-microbiomes"><span class="toc-section-number">1.2</span> Measuring Microbiomes</a></li>
<li><a href="#metrology"><span class="toc-section-number">1.3</span> Metrology</a></li>
</ul></li>
<li><a href="#s-rrna-gene"><span class="toc-section-number">2</span> 16S rRNA Gene</a><ul>
<li><a href="#general-background-about-16s-rrna-gene"><span class="toc-section-number">2.1</span> General background about 16S rRNA Gene</a></li>
<li><a href="#s-rrna-phylogenetics"><span class="toc-section-number">2.2</span> 16S rRNA phylogenetics</a></li>
<li><a href="#s-rrna-databases"><span class="toc-section-number">2.3</span> 16S rRNA Databases</a></li>
</ul></li>
<li><a href="#process-characterization"><span class="toc-section-number">3</span> Process Characterization</a></li>
<li><a href="#measurement-process"><span class="toc-section-number">4</span> Measurement Process</a><ul>
<li><a href="#sample-to-seq"><span class="toc-section-number">4.1</span> Sample to Seq</a><ul>
<li><a href="#sample-collection"><span class="toc-section-number">4.1.1</span> Sample Collection</a></li>
<li><a href="#sample-processing"><span class="toc-section-number">4.1.2</span> Sample Processing</a></li>
<li><a href="#s-rrna-pcr"><span class="toc-section-number">4.1.3</span> 16S rRNA PCR</a></li>
<li><a href="#library-preparation"><span class="toc-section-number">4.1.4</span> Library Preparation</a></li>
<li><a href="#sequencing"><span class="toc-section-number">4.1.5</span> Sequencing</a></li>
</ul></li>
<li><a href="#seq-to-otu"><span class="toc-section-number">4.2</span> Seq to OTU</a><ul>
<li><a href="#quality-filter-read-trimming"><span class="toc-section-number">4.2.1</span> quality filter/ read trimming</a></li>
<li><a href="#otu-assignment"><span class="toc-section-number">4.2.2</span> OTU Assignment</a></li>
<li><a href="#taxonomic-assignment"><span class="toc-section-number">4.2.3</span> Taxonomic Assignment</a></li>
</ul></li>
<li><a href="#downstream-analysis"><span class="toc-section-number">4.3</span> Downstream analysis</a><ul>
<li><a href="#differential-abundance"><span class="toc-section-number">4.3.1</span> Differential Abundance</a></li>
<li><a href="#alpha-diversity"><span class="toc-section-number">4.3.2</span> Alpha Diversity</a></li>
<li><a href="#beta-diversity"><span class="toc-section-number">4.3.3</span> Beta Diversity</a></li>
</ul></li>
<li><a href="#s-rrna-analysis-method-assessment"><span class="toc-section-number">4.4</span> 16S rRNA Analysis Method Assessment</a></li>
<li><a href="#sample-to-fastq"><span class="toc-section-number">4.5</span> Sample to fastq</a></li>
<li><a href="#seq-to-otu-1"><span class="toc-section-number">4.6</span> Seq to OTU</a><ul>
<li><a href="#pre-processing"><span class="toc-section-number">4.6.1</span> Pre-processing</a></li>
<li><a href="#chimera-detection"><span class="toc-section-number">4.6.2</span> Chimera Detection</a></li>
<li><a href="#otu-assignment-1"><span class="toc-section-number">4.6.3</span> OTU Assignment</a></li>
</ul></li>
<li><a href="#downstream-analysis-1"><span class="toc-section-number">4.7</span> Downstream analysis</a><ul>
<li><a href="#taxonomic-classification"><span class="toc-section-number">4.7.1</span> taxonomic classification</a></li>
<li><a href="#differential-abundance-1"><span class="toc-section-number">4.7.2</span> differential abundance</a></li>
<li><a href="#ecological-diversity"><span class="toc-section-number">4.7.3</span> ecological diversity</a></li>
</ul></li>
</ul></li>
<li><a href="#benchmarking-dataset"><span class="toc-section-number">5</span> Benchmarking Dataset</a></li>
<li><a href="#ecological-diversity-robustness"><span class="toc-section-number">6</span> Ecological Diversity Robustness</a><ul>
<li><a href="#overview"><span class="toc-section-number">6.1</span> Overview</a></li>
</ul></li>
<li><a href="#gold-standard-pipeline"><span class="toc-section-number">7</span> Gold Standard Pipeline</a></li>
<li><a href="#assessment-framework"><span class="toc-section-number">8</span> Assessment Framework</a></li>
<li><a href="#references"><span class="toc-section-number">9</span> References</a></li>
</ul>
<!--bookdown:toc2:end-->
      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Measurement Science of Metagenomics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<!--bookdown:toc:end-->
<!--bookdown:body:start-->
<div id="metrology-of-metagenomics" class="section level1">
<h1><span class="header-section-number">1</span> Metrology of Metagenomics</h1>
<div id="microbiome" class="section level2">
<h2><span class="header-section-number">1.1</span> Microbiome</h2>
</div>
<div id="measuring-microbiomes" class="section level2">
<h2><span class="header-section-number">1.2</span> Measuring Microbiomes</h2>
<ul>
<li>whole metagenomic sequencing<br />
</li>
<li>marker-gene metagenomics<br />
</li>
<li>16S vs. WMS</li>
</ul>
</div>
<div id="metrology" class="section level2">
<h2><span class="header-section-number">1.3</span> Metrology</h2>
<ul>
<li>No not meteorlogy<br />
</li>
<li>Biological variability and measurement error
<ul>
<li>Measurement Error
<ul>
<li>Systematic - results in systematic bias<br />
</li>
<li>Random - increases uncertainty in results but provides an accurate measure of the true mean given sufficient sample size<br />
</li>
</ul></li>
<li>Biological Noise
<ul>
<li>Sources of variability due to biological complexity of the system e.g. difference in results based on 16S rRNA gene region<br />
</li>
</ul></li>
</ul></li>
<li>replicate, repeat, and reproduce
<ul>
<li>SI redefining the kilogram example???<br />
</li>
<li>Uncertainty budgets
<ul>
<li>top-down vs. bottom-up<br />
</li>
<li>analytical chemsitry text<br />
</li>
</ul></li>
<li>Orthogonal methods
<ul>
<li>Genome in a bottle example</li>
</ul></li>
</ul></li>
</ul>
<!--

Remember each Rmd file contains one and only one chapter.

## Book Outline

## Metagenomics  
* Microbiome  
* Measuring Microbiomes 
    - whole metagenomic sequencing  
    - marker-gene metagenomics  
        + 16S rRNA gene  

## Marker-gene metagenomics measurement process  
* Measurement Process overview 
    - Sample collection  
    - Sample Processing  
    - Sequencing  
    - Data analysis  
        + fastq to OTU  
        + Downstream analysis  
            * differential abundance  
            * alpha diversity  
            * beta diversity  

## Biological variability and measurement error  
* Measurement Error  
    - Systematic - results in systematic bias  
    - Random - increases uncertainty in results but provides an accurate measure of the true mean given sufficient sample size  
* Biological Noise  
    - Sources of variability due to biological complexity of the system e.g. difference in results based on 16S rRNA gene region  
* replicate, repeat, and reproduce  
    - Uncertainty budgets  
        + top-down vs. bottom-up  
    - Orthogonal methods  

## Sources of variability and error for marker-gene metagenomics  
* fishbone diagram  
    - Sample collection  
    - Sample Processing  
        + DNA extraction  
        + PCR/ indexing  
        + Normalization/ Pooling  
    - Sequencing  
        + sequencing reaction  
        + imaging  
        + base calling  
    - Data analysis  
        + Pre-processing (wheat from the chaf)  
            * quality filter/ read trimming  
            * chimera detection  
            * clustering  
        + Taxonomic classification  
        + Downstream analysis  
            * differential abundance  
            * alpha diversity  
            * beta diversity  
* cause effect diagram  
    - pre-processing, normalization/ transformation methods used to address measurement error  

## Characterizing Measurement Process 
* Tools  
    - Validation  
        + Fit for purpose - comparison to gold standard method or reference material  
            * no gold standard methods - only community accepted defactor standards  
    - Benchmarking  
        + Comparison to known truth  
        + Evaluating results based on expectation  
    - Robustness testing  
        + Impact of procedure or step on results  
* Characterization of Data Analysis Processing  
    - Wet lab vs. dry lab  
    - Validation  
        + Currently not applicable ... -> most robust method  
    - Benchmarking  
        + Observed results vs. Truth  
            * Dataset with known true value - can be obtained using  
                - gold standard method  
                - mock communities: in vitro or in silico   
                    + limitations and assumptions  
                - Mixtures
                    + Provides a degree of known truth  
            * Performance metrics  
                - Quantitative  
                    + Variability - standard deviation/ variance  
                    + Bias - mean squared error??? (true value)  
        + Observed results vs. assumptions  
            * No general dataset requirements  
            * Performance metrics  
                - evaluate results based on method assumption, e.g. Schloss clustering distance  
    - Robustness Testing  
        + Used to evaluate impact of different methods on results  
        + Comparison of orthogonal methods  
            * For fundamentally different methods (different assumptions and heuristics)  
                - Can evaluate precision (variability), not necessarily accuracy but less susceptible to bias than is methods are based on the same assumptions or heuristics  
        + Used to evaluate how different steps in the process affect the results  

## Microbiome Mixture Benchmarking Dataset
* Mixture Study  

## Robustness Testing Ecological Diversity 
* Evaluating alpha and beta diversity metrics  
* Factors  
    - preprocessing methods  
    - data characteristics  
        + sequencing error rates  
        + primer region  
        + read length  
* OTU table  
    - depth/ coverage - sparsity  
    - when does the diversity metric break?  

## Guideance on developing an assessment framework
* Outline measurement process - fishbone diagram  
* Differentiate between biological variability and measurement error  
    - How are these two sources of variability currently addressed in pipelines  
    - Are there any known sources of biological variability not currently being addressed  
* What is known about each source of variability?  
    - uncertainty budget?  
    - dark uncertianty - unaccounted for variability?  
* Robustness testing  
    - determine largest source of bias/ uncertianty in measurement process  
* Charaterize individual measurement process steps based on robustness testing/ sensitivity analysis  
    - benchmarking datasets  
        + environmental  
        + in lab prepared material - combination of known samples  
        + in lab prepared material - combination of unknown samples  
        + in silico - various levels of simulation based on reference and environmental data availablility  
    - for each type of dataset summarize benefits and limitations to each  

## Proposed Gold Standard Analysis Method
* Trim primers - dereplicate  
* Chimera check? - multiple methods orthogonal methods?, annotate don't filter  
* BLAST 100% - all perfect matches  
* Unmatched  
    - Multiple sequence alignment  
        + Filter sequences with low alignment scores  
    - phylogenetic placement  
* Generate OTUs for different branch distances  
* Keep unique singletons - based on distance from non-singleton clusters, vary similarity based on cluster size, likelihood model for whether sequencing error or biological mutation.  
    - Rare orgs highly similar to large OTU less likely ecologically different?  
* Alternative taxa classification  
    - Multiple methods  
        + pairwise alignment  
        + kmer  
        + phylogenetic placement  
        + other...  
    - Multiple reference databases  
        + evaluate individual sequences based on confidence - presence in multiple databases, infernal and rnammer alignment score?  
            * supporting sequence data - whole genome, sanger, 16S PCR from isolate  
            * likelihood of being a chimera  
        + Compare other methods to results generated using gold standard pipeline  
            * performance metrics  
* Copy number correction  
    - based on phylogenetic placement
-->
<!--chapter:end:index.Rmd-->
</div>
</div>
<div id="s-rrna-gene" class="section level1">
<h1><span class="header-section-number">2</span> 16S rRNA Gene</h1>
<p>Not sure where this fits in…</p>
<div id="general-background-about-16s-rrna-gene" class="section level2">
<h2><span class="header-section-number">2.1</span> General background about 16S rRNA Gene</h2>
<ul>
<li>Biological function<br />
</li>
<li>Variable and conserved regions<br />
</li>
<li>Multicopy<br />
</li>
<li>Horizontal gene transfer<br />
</li>
<li>Gene diversity
<ul>
<li>secondary structure<br />
</li>
<li>Use of different variable regions</li>
</ul></li>
</ul>
</div>
<div id="s-rrna-phylogenetics" class="section level2">
<h2><span class="header-section-number">2.2</span> 16S rRNA phylogenetics</h2>
<ul>
<li>Gene evolution<br />
</li>
<li>Phylogeny</li>
</ul>
</div>
<div id="s-rrna-databases" class="section level2">
<h2><span class="header-section-number">2.3</span> 16S rRNA Databases</h2>
<ul>
<li>RDP<br />
</li>
<li>Greengenes<br />
</li>
<li>NCBI<br />
</li>
<li>SILVA</li>
</ul>
<!--chapter:end:00-16SrRNA-Gene.Rmd-->
</div>
</div>
<div id="process-characterization" class="section level1">
<h1><span class="header-section-number">3</span> Process Characterization</h1>
<ul>
<li>Tools
<ul>
<li>Validation
<ul>
<li>Fit for purpose - comparison to gold standard method or reference material
<ul>
<li>no gold standard methods - only community accepted defactor standards<br />
</li>
</ul></li>
</ul></li>
<li>Benchmarking
<ul>
<li>Comparison to known truth<br />
</li>
<li>Evaluating results based on expectation<br />
</li>
</ul></li>
<li>Robustness testing
<ul>
<li>Impact of procedure or step on results<br />
</li>
</ul></li>
</ul></li>
<li>Characterization of Data Analysis Processing
<ul>
<li>Wet lab vs. dry lab<br />
</li>
<li>Validation
<ul>
<li>Currently not applicable … -&gt; most robust method<br />
</li>
</ul></li>
<li>Benchmarking
<ul>
<li>Observed results vs. Truth
<ul>
<li>Dataset with known true value - can be obtained using
<ul>
<li>gold standard method<br />
</li>
<li>mock communities: in vitro or in silico
<ul>
<li>limitations and assumptions<br />
</li>
</ul></li>
<li>Mixtures
<ul>
<li>Provides a degree of known truth<br />
</li>
</ul></li>
</ul></li>
<li>Performance metrics
<ul>
<li>Quantitative
<ul>
<li>Variability - standard deviation/ variance<br />
</li>
<li>Bias - mean squared error??? (true value)<br />
</li>
</ul></li>
</ul></li>
</ul></li>
<li>Observed results vs. assumptions
<ul>
<li>No general dataset requirements<br />
</li>
<li>Performance metrics
<ul>
<li>evaluate results based on method assumption, e.g. Schloss clustering distance<br />
</li>
</ul></li>
</ul></li>
</ul></li>
<li>Robustness Testing
<ul>
<li>Used to evaluate impact of different methods on results<br />
</li>
<li>Comparison of orthogonal methods
<ul>
<li>For fundamentally different methods (different assumptions and heuristics)
<ul>
<li>Can evaluate precision (variability), not necessarily accuracy but less susceptible to bias than is methods are based on the same assumptions or heuristics<br />
</li>
</ul></li>
</ul></li>
<li>Used to evaluate how different steps in the process affect the results</li>
</ul></li>
</ul></li>
</ul>
<!--chapter:end:10-Assessment-Methods.Rmd-->
</div>
<div id="measurement-process" class="section level1">
<h1><span class="header-section-number">4</span> Measurement Process</h1>
<p>Overview of the 16S rRNA marker gene metagenomics measurement process including the sources of error and bias introduced at different step in the process along with different mitigation strategies that have been developed to address each of the issues.</p>
Diagram of measurement process @ref(fig:fishbone) .<br />

<div class="figure">
<img src="_main_files/figure-html/fishbone-1.png" alt="(\#fig:fishbone)Fishbone diagram of the 16S rRNA metagenomic sequencing measurement process."  />
<p class="caption">
(#fig:fishbone)Fishbone diagram of the 16S rRNA metagenomic sequencing measurement process.
</p>
</div>
<p>Review of what is currently known about the sources of error and bias in the 16S metagenomics measurement process. Cause-effect diagrams are commonly used in process quality control to show the relationship between steps in the measurement process and sources of error or bias @ref(fig:cause-effect).<br />
Potentially may want to revise to focus on sources of error, e.g. cell type bias, preferential amplification, sequence errors, chimeras .</p>
<div class="figure">
<img src="_main_files/figure-html/cause-effect-1.png" alt="(\#fig:cause-effect)Causes (branch labels) of bias and variability indicated for different sources of error (branch text)."  />
<p class="caption">
(#fig:cause-effect)Causes (branch labels) of bias and variability indicated for different sources of error (branch text).
</p>
</div>
<div id="sample-to-seq" class="section level2">
<h2><span class="header-section-number">4.1</span> Sample to Seq</h2>
<p>The first part of the measurement process is generating sequencing data from an environmental sample.</p>
<p>The part consists of five main steps:</p>
<ol style="list-style-type: decimal">
<li>Sample Collection<br />
</li>
<li>Sample Processing<br />
</li>
<li>16S rRNA PCR<br />
</li>
<li>Library Prep<br />
</li>
<li>Sequencing</li>
</ol>
<div id="sample-collection" class="section level3">
<h3><span class="header-section-number">4.1.1</span> Sample Collection</h3>
<ul>
<li>Collection of sample that is representative of population being studied.<br />
</li>
<li>Kitty Biome study as an example of sample heterogeneity.<br />
</li>
<li>Impact of sample storage.<br />
</li>
<li>Fecal storage study.<br />
</li>
<li>Old coral study?</li>
</ul>
</div>
<div id="sample-processing" class="section level3">
<h3><span class="header-section-number">4.1.2</span> Sample Processing</h3>
<ul>
<li>DNA extraction<br />
</li>
<li>Cell type specific extraction biases<br />
</li>
<li>DNA concentration requirements for library prep<br />
</li>
<li>Issues with low concentration samples - skin microbiome<br />
</li>
<li>Additional DNA purification to remove PCR inhibitors.<br />
</li>
<li>Old PCR inhibitor paper.<br />
</li>
<li>Any studies showing inhibitors inducing biases in metagenomic studies?
<ul>
<li>Potentially sample specific biases but do not know of any within sample biases</li>
</ul></li>
</ul>
</div>
<div id="s-rrna-pcr" class="section level3">
<h3><span class="header-section-number">4.1.3</span> 16S rRNA PCR</h3>
<ul>
<li>Target variable region<br />
</li>
<li>PCR primers (optional barcoded primers)</li>
<li>PCR reagents<br />
</li>
<li>Polymerase - high fidelity<br />
</li>
<li>Reagent contaminats<br />
</li>
<li>PCR cycling conditions<br />
</li>
<li>number of cycles</li>
<li>sequence copy abundance<br />
</li>
<li>chimeras</li>
</ul>
</div>
<div id="library-preparation" class="section level3">
<h3><span class="header-section-number">4.1.4</span> Library Preparation</h3>
<ul>
<li>Barcode/ indexing (optional)<br />
</li>
<li>Do not know of any barcode specific biases</li>
<li>Normalization and Pooling<br />
</li>
<li>Methods
<ul>
<li>DNA quantification<br />
</li>
<li>Bead based assays</li>
</ul></li>
<li>Associated Biases
<ul>
<li>Uneven sequencing depth - impact of coverage</li>
</ul></li>
</ul>
</div>
<div id="sequencing" class="section level3">
<h3><span class="header-section-number">4.1.5</span> Sequencing</h3>
<ul>
<li>Sequencing platform<br />
</li>
<li>Read length<br />
</li>
<li>Sequencing chemistry</li>
</ul>
</div>
</div>
<div id="seq-to-otu" class="section level2">
<h2><span class="header-section-number">4.2</span> Seq to OTU</h2>
<p>Convering raw sequence data to a OTU table with taxonomic annotations for use in downstream applications.</p>
<div id="quality-filter-read-trimming" class="section level3">
<h3><span class="header-section-number">4.2.1</span> quality filter/ read trimming</h3>
<ul>
<li>Filtering low quality reads - short reads with high error rates.<br />
</li>
<li>Trimming reads - removing ends of reads based on base quality score as well as removal of adapter sequences that were added during sample processing.</li>
<li>Merging paired ends</li>
<li>Merging forward and reverse reads into single contigs</li>
<li>chimera detection</li>
<li>detection and removal of chimeric reads, artifact of the PCR amplification process.</li>
</ul>
</div>
<div id="otu-assignment" class="section level3">
<h3><span class="header-section-number">4.2.2</span> OTU Assignment</h3>
<p>OTU assignment most commonly by clustering is used to reduce the number of spurious observations, sequences are grouped into OTUs.</p>
<ul>
<li>Different approaches to clustering
<ul>
<li>distance based clustering - most commonly used<br />
</li>
<li>Distance based clustering is a NP hard problem (i.e. computationaly intense), therefore a number of heuristics are used to make the method computationally tractable for large datasets.</li>
<li>Threshold are used to define the similarity of sequences within a cluster, 99% and 97% are commonly used.
<ul>
<li>Some biological and sequencing error basis for threshold values, (e.g. 99% species level, 97% genus level, 1% error rate). Taxonomic variability in 16S rRNA sequence diversity (Pai?) and sequencing error <span class="citation">(Huse et al. <a href="#ref-Huse2010">2010</a>, <span class="citation">Schirmer et al. (<a href="#ref-Schirmer2015">2015</a>)</span>, <span class="citation">Amore et al. (<a href="#ref-Amore2016">2016</a>)</span>)</span> challenges the validity of these values.</li>
</ul></li>
<li>Different distance based thresholds.
<ul>
<li>average linkage<br />
</li>
<li>single linkage<br />
</li>
<li>complete linkage<br />
</li>
</ul></li>
<li>Other clustering/ OTU assignment methods include:</li>
<li>distribution based<br />
</li>
<li>oligotyping<br />
</li>
<li>tree based clustering <span class="citation">(Wu, Doroud, and Eisen <a href="#ref-Wu2013">2013</a>)</span></li>
<li>De novo vs Reference based clustering</li>
<li>Sequences can be clustered with a set of reference cluster centers or without (<em>de novo</em>).</li>
</ul></li>
</ul>
</div>
<div id="taxonomic-assignment" class="section level3">
<h3><span class="header-section-number">4.2.3</span> Taxonomic Assignment</h3>
<p>Assign clusters a taxonomic name</p>
</div>
</div>
<div id="downstream-analysis" class="section level2">
<h2><span class="header-section-number">4.3</span> Downstream analysis</h2>
<div id="differential-abundance" class="section level3">
<h3><span class="header-section-number">4.3.1</span> Differential Abundance</h3>
<ul>
<li>Differences in taxa or OTU abundance between experimental factors, e.g. treatment vs. control.</li>
</ul>
</div>
<div id="alpha-diversity" class="section level3">
<h3><span class="header-section-number">4.3.2</span> Alpha Diversity</h3>
<ul>
<li>Measure of biological or ecological diversity within a sample.<br />
</li>
<li>Alpha diversity as a continuum related to the metric dependence on richness and evenness <span class="citation">(Chao et al. <a href="#ref-Chao2014">2014</a>)</span></li>
<li>Four primary categories richness, evenness, abundance weighted, and dissimilarity <span class="citation">(Ricotta <a href="#ref-Ricotta2007">2007</a>)</span></li>
<li>Richness
<ul>
<li>Hill numbers, rarefaction, and richness estimation <span class="citation">(Chao et al. <a href="#ref-Chao2014">2014</a>)</span></li>
</ul></li>
<li>Evenness
<ul>
<li>Criteria for evaluating evenness metrics <span class="citation">(Tuomisto <a href="#ref-Tuomisto2012">2012</a>)</span></li>
<li>See <span class="citation">Tuomisto (<a href="#ref-Tuomisto2012">2012</a>)</span> Table 1 for summary of evenness metric including interpretation</li>
</ul></li>
<li>Abundance weighted</li>
<li>Dissimilarity</li>
<li>Parametric vs. Non-parametric diversity metrics for 16S rRNA sequence analysis <span class="citation">(Bunge, Willis, and Walsh <a href="#ref-Bunge2014">2014</a>)</span></li>
<li>Development of alpha diversity richness metric for microbial diversity.<br />
</li>
<li>Chao1 a traditional diversity metric estimates total diversity using the number of singletons, OTUs representing only a single sequence.
<ul>
<li>Issue with singleton based diversity metrics for microbial diversity is that for incresed sequencing depth the number of singletons with continue to increase due to sequencing errors, leading to artifically inflated diversity estimates.</li>
</ul></li>
<li><span class="citation">Chiu and Chao (<a href="#ref-Chiu2015">2015</a>)</span> developed a modified version of the Chao1 estimator that is more robust to singletons bue to sequencing errors.</li>
<li>This method estimates diversity based on the number of doubletons to tentons.<br />
</li>
<li>Incorporates Hill numbers as well.</li>
</ul>
</div>
<div id="beta-diversity" class="section level3">
<h3><span class="header-section-number">4.3.3</span> Beta Diversity</h3>
<!--chapter:end:20-Measurement-Process.Rmd-->
</div>
</div>
<div id="s-rrna-analysis-method-assessment" class="section level2">
<h2><span class="header-section-number">4.4</span> 16S rRNA Analysis Method Assessment</h2>
<p>Method steps<br />
1. Sample to Seq 1. Seq to OTU<br />
2. Downstream analysis 3. Taxonomic Classification<br />
4. Differential Abundance<br />
5. Ecological Diversity</p>
</div>
<div id="sample-to-fastq" class="section level2">
<h2><span class="header-section-number">4.5</span> Sample to fastq</h2>
<ul>
<li>Benchmarking
<ul>
<li>using <em>in vitro</em> mock communities (e.g. cells, DNA, or PCR products)<br />
</li>
<li>technical replicates?<br />
</li>
<li>well characterized benchmarking samples - similar to Genomic RMs (deep sequenced multiple methods)<br />
</li>
<li>Mixture/ titrations<br />
</li>
<li>Standard additions<br />
</li>
<li>Dilution to extinction?</li>
</ul></li>
</ul>
</div>
<div id="seq-to-otu-1" class="section level2">
<h2><span class="header-section-number">4.6</span> Seq to OTU</h2>
<div id="pre-processing" class="section level3">
<h3><span class="header-section-number">4.6.1</span> Pre-processing</h3>
<p>Schloss, P. D. (2010). The Effects of Alignment Quality, Distance Calculation Method, Sequence Filtering, and Region on the Analysis of 16S rRNA Gene-Based Studies. PLoS Comput Biol, 6(7), e1000844. <a href="http://doi.org/10.1371/journal.pcbi.1000844" class="uri">http://doi.org/10.1371/journal.pcbi.1000844</a></p>
<p>Albanese, D., Fontana, P., De Filippo, C., Cavalieri, D., &amp; Donati, C. (2015). MICCA: a complete and accurate software for taxonomic profiling of metagenomic data. Scientific Reports, 5, 9743. <a href="http://doi.org/10.1038/srep09743" class="uri">http://doi.org/10.1038/srep09743</a><br />
* Evaluates preprocessing % read passing filtering, chimeras, and redundant OTUs, relative abundance</p>
<p>Gaspar, J. M., &amp; Thomas, W. K. (2013). Assessing the consequences of denoising marker-based metagenomic data. PloS One, 8(3), e60458. <a href="http://doi.org/10.1371/journal.pone.0060458" class="uri">http://doi.org/10.1371/journal.pone.0060458</a><br />
* Evaluation of denoising and chimera detection methods for 454 data</p>
</div>
<div id="chimera-detection" class="section level3">
<h3><span class="header-section-number">4.6.2</span> Chimera Detection</h3>
<p>Mysara, M., Saeys, Y., Leys, N., Raes, J., &amp; Monsieurs, P. (2015). CATCh, an Ensemble Classifier for Chimera Detection in 16S rRNA Sequencing Studies. Applied and Environmental Microbiology, 81(December), 1573–1584. <a href="http://doi.org/10.1128/AEM.02896-14" class="uri">http://doi.org/10.1128/AEM.02896-14</a></p>
<p>Beblur - <a href="https://github.com/ekopylova/deblur" class="uri">https://github.com/ekopylova/deblur</a></p>
</div>
<div id="otu-assignment-1" class="section level3">
<h3><span class="header-section-number">4.6.3</span> OTU Assignment</h3>
<ul>
<li>De novo clustering has been shown to be more accurate <span class="citation">(S. L. Westcott and Schloss <a href="#ref-Westcott2015">2015</a>)</span><br />
</li>
<li>Types of datasets used to assess benchmarking methods: simulated data, mock communities, environmental datasests</li>
<li>Approaches to evaluating clustering methods
<ul>
<li>Cluster stability - similarity in clustering results when varying input data<br />
</li>
<li>Cluster reproducibility - similarity in clustering results between methods<br />
</li>
<li>Number of artifact OTUs - number of observed OTUs relative to expected
<ul>
<li>Evaluating clustering methods using simulated and mock community clustering data provide a level of truth (expectation of the correct result).</li>
</ul></li>
</ul></li>
</ul>
<div id="summary-of-cluster-method-assessment-papers" class="section level4">
<h4><span class="header-section-number">4.6.3.1</span> Summary of Cluster Method Assessment Papers</h4>
<ul>
<li><span class="citation">Kopylova et al. (<a href="#ref-Kopylova2014">2014</a>)</span> used a combination of simulated, mock communities, and data from environmental samples to compare clustering methods.
<ul>
<li>They evaluated the performance of the clustering methods using F-measure and phylogenetic distance.<br />
</li>
<li>For environment datasets the clustering methods were evaluated based on the disimilarity of samples for a dataset based on the sum of the squared deviation for UniFrac PCoA (Procrustes M^2) and similarity to UCLUST (Pearson’s correlation). <strong>NOTE</strong> Still note sure what they are evaluating for Procrustes.<br />
</li>
<li>The authors excluded singletons from there analysis, while this does help to eliminate spurious OTUs, the method may also exclude rare taxa.<br />
</li>
</ul></li>
<li><span class="citation">Schloss (<a href="#ref-Schloss2016">2016</a>)</span> argues that using simulated and mock communities is not representative of real environmental samples. The authors additionally, argue the approach used by <span class="citation">Kopylova et al. (<a href="#ref-Kopylova2014">2014</a>)</span> to evaluating clustering methods confounds the impact of read filtering with clustering methods.
<ul>
<li>The author’s evaluated environmental datasets using Mathew’s correlation coefficient, for a truth table (TP, TN, FP, FN) based on distance between sequences in the same and different clusters.
<ul>
<li>This method for evaluating clusters was first used in <span class="citation">(P. D. Schloss and Westcott <a href="#ref-Schloss2011">2011</a>)</span><br />
</li>
</ul></li>
<li>Average neighbor performed the best based the MCC based assessment. However, this approach to evaluation is biased towards this method as the results are strictly assessed based on the sequence distance criteria. Other clustering methods that attempt to address more nuianced issues with clustering, sequencing error and variability in sequence diversity allow for differences in distances within and between clusters.<br />
</li>
<li>This approach to cluster evaluation is particularly suited for algorithms based on analytical proofs.</li>
<li>For example the DNACluster clustering algorithm employs a heuristic based on a proof that for any given cluster the distance between two sequences in the cluster is less than the defined threshold value and no sequence is closer to another cluster center than the center of the cluster it is assigned to <span class="citation">(Ghodsi, Liu, and Pop <a href="#ref-Ghodsi2011">2011</a>)</span>.
<ul>
<li><span class="citation">Ghodsi, Liu, and Pop (<a href="#ref-Ghodsi2011">2011</a>)</span> used a similar approach to validate their clustering method, though only focusing on the within cluster pairwise distance.</li>
</ul></li>
</ul></li>
<li>Cluster Robustness is another attribute for evaluating clustering methods <span class="citation">(He et al. <a href="#ref-He2015">2015</a>)</span>.
<ul>
<li>Cluster robustness is the reproduciblity of the OTU table generated by a clustering algorith.<br />
</li>
<li>Cluster robustness was evaluated based on:
<ul>
<li>Unstable sequences - sequences represented by different centroids</li>
<li>Unstable OTUs - OTUs where membership was impacted by the number of sequences in a dataset.</li>
<li>using Mathew’s correlation coefficient
<ul>
<li>Truth Table definitions
<ul>
<li>TP if two sequences clustered <strong>together</strong> for the full and subsampled dataset<br />
</li>
<li>TN if two sequences clustered <strong>separately</strong> for the full and subsampled<br />
</li>
<li>FN two sequences clustered <strong>together</strong> in full but not subsampled.<br />
</li>
<li>FP two sequences clustered <strong>separately</strong> in full but together in subsampled dataset.</li>
</ul></li>
<li>The MCC method uses the full dataset at the true value to benchmark the subsampled dataset against.<br />
</li>
<li><strong>NOTE</strong> Is this a valid assumption for clustering?<br />
</li>
</ul></li>
<li>Impact of cluster stablity on Rarefaction curves, PCoA with Bray-Curtis and UniFrac.<br />
</li>
<li>Tested for significance using ADNOIS, a non-parametric test measuring effect size, the amount of the observed variance explained by metadata variables. <strong>NEED REF</strong> - See vegan manual for potenial references.</li>
</ul></li>
<li>The resulting OTU table is dependent the number of sequences clustered.<br />
</li>
<li>For some methods the OTU table is dependent on the order in which the sequences are provided <span class="citation">(Ghodsi, Liu, and Pop <a href="#ref-Ghodsi2011">2011</a>)</span>.<br />
</li>
<li>Closed reference clustering was the only method the produced stable clusters.<br />
</li>
<li>This method discards sequences that do not cluster with reference sequences.<br />
</li>
<li>Open-reference cluster, which performs <em>de novo</em> clustering of unmatched sequences provides a more robust alternative clustering method to <em>de novo</em> clustering while allowing for the diversity of novel OTUs.<br />
</li>
</ul></li>
<li>Cluster method robustness can also be calculated using cross validation<span class="citation">(Chen et al. <a href="#ref-Chen2013">2013</a>)</span>.
<ul>
<li><span class="citation">Chen et al. (<a href="#ref-Chen2013">2013</a>)</span> subsampled the data to 90% five times and compared the clusters between replicates.<br />
</li>
<li>Methods were evaluated using precision, recall, and NID
<ul>
<li>precision - measure of cluster homogeneity<br />
</li>
<li>recall - measure of cluster completeness<br />
</li>
<li>NID - assess cluster globally???<br />
</li>
</ul></li>
<li>See assessment of cluster quality for metric definitions<br />
</li>
</ul></li>
<li>Cluster robustness can also be evaluated based on threshold and 16S region <span class="citation">(Schmidt, Matias Rodrigues, and Mering <a href="#ref-Schmidt2015">2015</a>)</span>.
<ul>
<li>The authors focus their assessment more on the comparability of results among different methods and note the importances of reproducibility.<br />
</li>
<li>Quote from paper “how robust are biological findings to the choice of clustering method? We found that OTU demarcation may indeed be replicable: different methods provided (almost) identical partitions when twice clustering the exact same sets of sequences, but in randomized order (Fig. 4, diagonals). However, trends in reproducibility were less clear.”<br />
</li>
</ul></li>
<li>Cluster method assessment based on OTU inflation.
<ul>
<li>Clustering methods have also been evaluated based on the number of predicted OTUs relative to the number of expected OTUs <span class="citation">(Huse et al. <a href="#ref-Huse2010">2010</a>, <span class="citation">Kopylova et al. (<a href="#ref-Kopylova2014">2014</a>)</span>)</span>.<br />
</li>
<li>Here either simulated or data from a mock community, where the number of OTUs is based on the number of strains or genomes used to generate the assessment dataset.<br />
</li>
<li>The limitation to this approach is that it does not account for contaminants when <strong>in vitro</strong> mock communities are used, and neither <strong>in silico</strong> or <strong>in vitro</strong> datasets represent the true complexity and diversity of environmental samples.</li>
</ul></li>
</ul>
</div>
</div>
</div>
<div id="downstream-analysis-1" class="section level2">
<h2><span class="header-section-number">4.7</span> Downstream analysis</h2>
<div id="taxonomic-classification" class="section level3">
<h3><span class="header-section-number">4.7.1</span> taxonomic classification</h3>
</div>
<div id="differential-abundance-1" class="section level3">
<h3><span class="header-section-number">4.7.2</span> differential abundance</h3>
</div>
<div id="ecological-diversity" class="section level3">
<h3><span class="header-section-number">4.7.3</span> ecological diversity</h3>
<!--chapter:end:30-Method-Assessment.Rmd-->
</div>
</div>
</div>
<div id="benchmarking-dataset" class="section level1">
<h1><span class="header-section-number">5</span> Benchmarking Dataset</h1>
<ul>
<li>Mixture Study</li>
</ul>
<!--chapter:end:40-Mixture-Study.Rmd-->
</div>
<div id="ecological-diversity-robustness" class="section level1">
<h1><span class="header-section-number">6</span> Ecological Diversity Robustness</h1>
<div id="overview" class="section level2">
<h2><span class="header-section-number">6.1</span> Overview</h2>
<ul>
<li>Background</li>
<li>Evaluating alpha and beta diversity metrics<br />
</li>
<li>Factors
<ul>
<li>preprocessing methods<br />
</li>
<li>data characteristics
<ul>
<li>sequencing error rates<br />
</li>
<li>primer region<br />
</li>
<li>read length<br />
</li>
</ul></li>
</ul></li>
<li>OTU table
<ul>
<li>depth/ coverage - sparsity<br />
</li>
<li>when does the diversity metric break?</li>
</ul></li>
</ul>
<!--chapter:end:50-Diversity-Metric-Robustness.Rmd-->
</div>
</div>
<div id="gold-standard-pipeline" class="section level1">
<h1><span class="header-section-number">7</span> Gold Standard Pipeline</h1>
<ul>
<li>Trim primers - dereplicate<br />
</li>
<li>Chimera check? - multiple methods orthogonal methods?, annotate don’t filter<br />
</li>
<li>BLAST 100% - all perfect matches<br />
</li>
<li>Unmatched
<ul>
<li>Multiple sequence alignment
<ul>
<li>Filter sequences with low alignment scores<br />
</li>
</ul></li>
<li>phylogenetic placement<br />
</li>
</ul></li>
<li>Generate OTUs for different branch distances<br />
</li>
<li>Keep unique singletons - based on distance from non-singleton clusters, vary similarity based on cluster size, likelihood model for whether sequencing error or biological mutation.
<ul>
<li>Rare orgs highly similar to large OTU less likely ecologically different?<br />
</li>
</ul></li>
<li>Alternative taxa classification
<ul>
<li>Multiple methods
<ul>
<li>pairwise alignment<br />
</li>
<li>kmer<br />
</li>
<li>phylogenetic placement<br />
</li>
<li>other…<br />
</li>
</ul></li>
<li>Multiple reference databases
<ul>
<li>evaluate individual sequences based on confidence - presence in multiple databases, infernal and rnammer alignment score?
<ul>
<li>supporting sequence data - whole genome, sanger, 16S PCR from isolate<br />
</li>
<li>likelihood of being a chimera<br />
</li>
</ul></li>
<li>Compare other methods to results generated using gold standard pipeline
<ul>
<li>performance metrics<br />
</li>
</ul></li>
</ul></li>
</ul></li>
<li>Copy number correction
<ul>
<li>based on phylogenetic placement</li>
</ul></li>
</ul>
<!--chapter:end:60-Golden-Pipeline.Rmd-->
</div>
<div id="assessment-framework" class="section level1">
<h1><span class="header-section-number">8</span> Assessment Framework</h1>
<ul>
<li><p>Outline measurement process - fishbone diagram</p></li>
<li>Differentiate between biological variability and measurement error
<ul>
<li>How are these two sources of variability currently addressed in pipelines</li>
<li>Are there any known sources of biological variability not currently being addressed</li>
</ul></li>
<li>What is known about each source of variability?
<ul>
<li>uncertainty budget?<br />
</li>
<li>dark uncertianty - unaccounted for variability?</li>
</ul></li>
<li>Robustness testing
<ul>
<li>determine largest source of bias/ uncertianty in measurement process<br />
</li>
<li>Schmidt 2014 Repeatability and Reproducibility</li>
<li>Evaluation - no gold standard method, or known truth
<ul>
<li>Quantifying similarity in results
<ul>
<li>Schmidt et al 2014 Env Micro</li>
<li>Schloss et al 2016</li>
</ul></li>
<li>Method with similar assumptions</li>
<li>Methods with different assumption</li>
</ul></li>
</ul></li>
<li>Charaterize individual measurement process steps based on robustness testing/ sensitivity analysis
<ul>
<li>benchmarking datasets
<ul>
<li>environmental<br />
</li>
<li>in lab prepared material - combination of known samples<br />
</li>
<li>in lab prepared material - combination of unknown samples<br />
</li>
<li>in silico - various levels of simulation based on reference and environmental data availablility<br />
</li>
</ul></li>
<li>for each type of dataset summarize benefits and limitations to each</li>
</ul></li>
<li>Evaluation Framework
<ul>
<li>Requirements
<ul>
<li>Benchmarking dataset<br />
</li>
<li>Metric for evaluating performance
<ul>
<li>Assumption Based vs. Emerical Assessment
<ul>
<li>Schloss et al. paper using within and between cluster distances to benchmark clustering methods<br />
</li>
</ul></li>
<li>Thinking about what is being measured - biological question</li>
</ul></li>
</ul></li>
</ul></li>
<li>Comparison of methods
<ul>
<li>Benchmarking
<ul>
<li>Gold standard method - known correct answer
<ul>
<li>For new method if different how to determine if new method is better?<br />
</li>
</ul></li>
<li>Reference dataset - known truth<br />
</li>
</ul></li>
<li>Evaluation Methods
<ul>
<li>Benchmarking
<ul>
<li>Comparing results to known truth
<ul>
<li>limitation with biological applications no real ground truth.
<ul>
<li>For physical and chemical metrology a ground truth is determined through uncertainty analysis with traceability to the SI. Often requiring the use of two orthogonal methods and uncertainty budgets. Currently no methods for traceing sequence data to an SI.<br />
</li>
</ul></li>
</ul></li>
<li>Benchmarking datasets are generated from materials or <em>in silico</em> with a level of confidence regarding expected results.
<ul>
<li>Discussion of top down vs. bottom up uncertainty analysis for use in establishing truth with benchmarking datasets.<br />
</li>
</ul></li>
<li>Results from analysis of benchmarking datasets evaluated using performance metrics, e.g. truth tables for qualitative data.<br />
</li>
</ul></li>
<li>Benchmarking- optimization
<ul>
<li>evaluating method performance when subjected to changes in either quality of data (increased noise) or use of different methods or parameters (optimization) similar to sensitivity analysis but comparing results to a dataset with known truth.</li>
</ul></li>
</ul></li>
</ul></li>
</ul>
<!--chapter:end:70-Assessment-Framework.Rmd-->
</div>
<div id="references" class="section level1">
<h1><span class="header-section-number">9</span> References</h1>
<!--chapter:end:80-References.Rmd-->
<div id="refs" class="references">
<div id="ref-Amore2016">
<p>Amore, Rosalinda D, Umer Zeeshan Ijaz, Melanie Schirmer, John G Kenny, Richard Gregory, Alistair C Darby, Christopher Quince, et al. 2016. “A comprehensive benchmarking study of protocols and sequencing platforms for 16S rRNA community profiling.” <em>BMC Genomics</em> 17. BMC Genomics: 1–40. doi:<a href="https://doi.org/10.1186/s12864-015-2194-9">10.1186/s12864-015-2194-9</a>.</p>
</div>
<div id="ref-Bunge2014">
<p>Bunge, John, Amy Willis, and Fiona Walsh. 2014. “Estimating the Number of Species in Microbial Diversity Studies.” <em>Annual Review of Statistics and Its Application</em> 1 (1): 427–45. doi:<a href="https://doi.org/10.1146/annurev-statistics-022513-115654">10.1146/annurev-statistics-022513-115654</a>.</p>
</div>
<div id="ref-Chao2014">
<p>Chao, Anne, Nicholas J. Gotelli, T. C. Hsieh, Elizabeth L. Sander, K. H. Ma, Robert K. Colwell, and Aaron M. Ellison. 2014. “Rarefaction and extrapolation with Hill numbers: A framework for sampling and estimation in species diversity studies.” <em>Ecological Monographs</em> 84 (1): 45–67. doi:<a href="https://doi.org/10.1890/13-0133.1">10.1890/13-0133.1</a>.</p>
</div>
<div id="ref-Chen2013">
<p>Chen, Wei, Clarence K. Zhang, Yongmei Cheng, Shaowu Zhang, and Hongyu Zhao. 2013. “A Comparison of Methods for Clustering 16S rRNA Sequences into OTUs.” <em>PLoS ONE</em> 8 (8). doi:<a href="https://doi.org/10.1371/journal.pone.0070837">10.1371/journal.pone.0070837</a>.</p>
</div>
<div id="ref-Chiu2015">
<p>Chiu, Chun-Huo, and Anne Chao. 2015. “Estimating and comparing microbial diversity in the presence of sequencing errors.” <em>PeerJ</em>, 1–13. doi:<a href="https://doi.org/10.7287/peerj.preprints.11">10.7287/peerj.preprints.11</a>.</p>
</div>
<div id="ref-Ghodsi2011">
<p>Ghodsi, Mohammadreza, Bo Liu, and Mihai Pop. 2011. “DNACLUST: accurate and efficient clustering of phylogenetic marker genes.” <em>BMC Bioinformatics</em> 12 (1). BioMed Central Ltd: 271. doi:<a href="https://doi.org/10.1186/1471-2105-12-271">10.1186/1471-2105-12-271</a>.</p>
</div>
<div id="ref-He2015">
<p>He, Yan, J Gregory Caporaso, Xiao-Tao Jiang, Hua-Fang Sheng, Susan M Huse, Jai Ram Rideout, Robert C Edgar, et al. 2015. “Stability of operational taxonomic units: an important but neglected property for analyzing microbial diversity.” <em>Microbiome</em> 3 (1). ??? 20. doi:<a href="https://doi.org/10.1186/s40168-015-0081-x">10.1186/s40168-015-0081-x</a>.</p>
</div>
<div id="ref-Huse2010">
<p>Huse, Susan M, David Mark Welch, Hilary G Morrison, and Mitchell L Sogin. 2010. “Ironing out the wrinkles in the rare biosphere through improved OTU clustering.” <em>Environmental Microbiology</em> 12 (7): 1889–98. doi:<a href="https://doi.org/10.1111/j.1462-2920.2010.02193.x">10.1111/j.1462-2920.2010.02193.x</a>.</p>
</div>
<div id="ref-Kopylova2014">
<p>Kopylova, Evguenia, Jose A Navas-molina, Céline Mercier, and Zech Xu. 2014. “Open-Source Sequence Clustering Methods Improve the State Of the Art.” <em>MSystems</em> 1 (1): 1–16. doi:<a href="https://doi.org/10.1128/mSystems.00003-15.Editor">10.1128/mSystems.00003-15.Editor</a>.</p>
</div>
<div id="ref-Ricotta2007">
<p>Ricotta, Carlo. 2007. “A semantic taxonomy for diversity measures.” <em>Acta Biotheoretica</em> 55 (1): 23–33. doi:<a href="https://doi.org/10.1007/s10441-007-9008-7">10.1007/s10441-007-9008-7</a>.</p>
</div>
<div id="ref-Schirmer2015">
<p>Schirmer, Melanie, Umer Z. Ijaz, Rosalinda D’Amore, Neil Hall, William T. Sloan, and Christopher Quince. 2015. “Insight into biases and sequencing errors for amplicon sequencing with the Illumina MiSeq platform.” <em>Nucleic Acids Research</em> 43 (6). doi:<a href="https://doi.org/10.1093/nar/gku1341">10.1093/nar/gku1341</a>.</p>
</div>
<div id="ref-Schloss2016">
<p>Schloss, Patrick D. 2016. “Application of database-independent approach to assess the quality of OTU picking methods.”</p>
</div>
<div id="ref-Schloss2011">
<p>Schloss, Patrick D., and Sarah L. Westcott. 2011. “Assessing and Improving Methods Used in Operational Taxonomic Unit-Based Approaches for 16S rRNA Gene Sequence Analysis.” <em>Applied and Environmental Microbiology</em> 77 (10): 3219–26. doi:<a href="https://doi.org/10.1128/AEM.02810-10">10.1128/AEM.02810-10</a>.</p>
</div>
<div id="ref-Schmidt2015">
<p>Schmidt, Thomas S B, Jo??o F. Matias Rodrigues, and Christian von Mering. 2015. “Limits to robustness and reproducibility in the demarcation of operational taxonomic units.” <em>Environmental Microbiology</em> 17 (5): 1689–1706. doi:<a href="https://doi.org/10.1111/1462-2920.12610">10.1111/1462-2920.12610</a>.</p>
</div>
<div id="ref-Tuomisto2012">
<p>Tuomisto, Hanna. 2012. “An updated consumer’s guide to evenness and related indices.” <em>Oikos</em> 121 (8): 1203–18. doi:<a href="https://doi.org/10.1111/j.1600-0706.2011.19897.x">10.1111/j.1600-0706.2011.19897.x</a>.</p>
</div>
<div id="ref-Westcott2015">
<p>Westcott, Sarah L, and Patrick D Schloss. 2015. “De novo clustering methods out-perform reference-based methods for assigning 16S rRNA gene sequences to operational taxonomic units.” <em>PeerJ</em> 3: e1487. doi:<a href="https://doi.org/10.7287/peerj.preprints.1466v1">10.7287/peerj.preprints.1466v1</a>.</p>
</div>
<div id="ref-Wu2013">
<p>Wu, Dongying, Ladan Doroud, and Jonathan a. Eisen. 2013. “TreeOTU Operational Taxonomic Unit Classification Based on Phylogenetic Trees.” <em>Arxiv.Org</em>, 23. <a href="http://arxiv.org/abs/1308.6333" class="uri">http://arxiv.org/abs/1308.6333</a>.</p>
</div>
</div>
</div>
<!--bookdown:body:end-->
            </section>

          </div>
        </div>
      </div>
<!--bookdown:link_prev-->
<!--bookdown:link_next-->

<!--bookdown:config-->


</body>

</html>
