<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Measurement Science of Metagenomics</title>
  <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
  <meta name="description" content="Measurement Science of Metagenomics">
  <meta name="generator" content="bookdown 0.0.52 and GitBook 2.6.7">



<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  TeX: { equationNumbers: { autoNumber: "AMS" } }
});
</script>
  <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="gold-standard-pipeline.html">
<link rel="next" href="references.html">

<link href="libs/gitbook/css/style.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>


  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Metrology of Marker-Gene Metagenomics </a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Metrology of Metagenomics</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#microbiome"><i class="fa fa-check"></i><b>1.1</b> Microbiome</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#measuring-microbiomes"><i class="fa fa-check"></i><b>1.2</b> Measuring Microbiomes</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#metrology"><i class="fa fa-check"></i><b>1.3</b> Metrology</a></li>
<li class="chapter" data-level="1.4" data-path="index.html"><a href="index.html#key-references"><i class="fa fa-check"></i><b>1.4</b> Key references</a><ul>
<li class="chapter" data-level="1.4.1" data-path="index.html"><a href="index.html#measurement-process"><i class="fa fa-check"></i><b>1.4.1</b> Measurement Process</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html"><i class="fa fa-check"></i><b>2</b> 16S rRNA Gene</a><ul>
<li class="chapter" data-level="2.1" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html#general-background-about-16s-rrna-gene"><i class="fa fa-check"></i><b>2.1</b> General background about 16S rRNA Gene</a></li>
<li class="chapter" data-level="2.2" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html#s-rrna-phylogenetics"><i class="fa fa-check"></i><b>2.2</b> 16S rRNA phylogenetics</a></li>
<li class="chapter" data-level="2.3" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html#s-rrna-databases"><i class="fa fa-check"></i><b>2.3</b> 16S rRNA Databases</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="process-characterization.html"><a href="process-characterization.html"><i class="fa fa-check"></i><b>3</b> Process Characterization</a></li>
<li class="chapter" data-level="4" data-path="measurement-process-1.html"><a href="measurement-process-1.html"><i class="fa fa-check"></i><b>4</b> Measurement Process</a><ul>
<li class="chapter" data-level="4.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-to-seq"><i class="fa fa-check"></i><b>4.1</b> Sample to Seq</a><ul>
<li class="chapter" data-level="4.1.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-collection"><i class="fa fa-check"></i><b>4.1.1</b> Sample Collection</a></li>
<li class="chapter" data-level="4.1.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-processing"><i class="fa fa-check"></i><b>4.1.2</b> Sample Processing</a></li>
<li class="chapter" data-level="4.1.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#s-rrna-pcr"><i class="fa fa-check"></i><b>4.1.3</b> 16S rRNA PCR</a></li>
<li class="chapter" data-level="4.1.4" data-path="measurement-process-1.html"><a href="measurement-process-1.html#library-preparation"><i class="fa fa-check"></i><b>4.1.4</b> Library Preparation</a></li>
<li class="chapter" data-level="4.1.5" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sequencing"><i class="fa fa-check"></i><b>4.1.5</b> Sequencing</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#seq-to-otu"><i class="fa fa-check"></i><b>4.2</b> Seq to OTU</a><ul>
<li class="chapter" data-level="4.2.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#quality-filter-read-trimming"><i class="fa fa-check"></i><b>4.2.1</b> quality filter/ read trimming</a></li>
<li class="chapter" data-level="4.2.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#otu-assignment"><i class="fa fa-check"></i><b>4.2.2</b> OTU Assignment</a></li>
<li class="chapter" data-level="4.2.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#taxonomic-assignment"><i class="fa fa-check"></i><b>4.2.3</b> Taxonomic Assignment</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#downstream-analysis"><i class="fa fa-check"></i><b>4.3</b> Downstream analysis</a><ul>
<li class="chapter" data-level="4.3.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#differential-abundance"><i class="fa fa-check"></i><b>4.3.1</b> Differential Abundance</a></li>
<li class="chapter" data-level="4.3.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#alpha-diversity"><i class="fa fa-check"></i><b>4.3.2</b> Alpha Diversity</a></li>
<li class="chapter" data-level="4.3.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#beta-diversity"><i class="fa fa-check"></i><b>4.3.3</b> Beta Diversity</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="measurement-process-1.html"><a href="measurement-process-1.html#step-by-step-protocols"><i class="fa fa-check"></i><b>4.4</b> Step-by-Step protocols</a></li>
<li class="chapter" data-level="4.5" data-path="measurement-process-1.html"><a href="measurement-process-1.html#s-rrna-analysis-method-assessment"><i class="fa fa-check"></i><b>4.5</b> 16S rRNA Analysis Method Assessment</a></li>
<li class="chapter" data-level="4.6" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-to-fastq"><i class="fa fa-check"></i><b>4.6</b> Sample to fastq</a></li>
<li class="chapter" data-level="4.7" data-path="measurement-process-1.html"><a href="measurement-process-1.html#seq-to-otu-1"><i class="fa fa-check"></i><b>4.7</b> Seq to OTU</a><ul>
<li class="chapter" data-level="4.7.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#pre-processing"><i class="fa fa-check"></i><b>4.7.1</b> Pre-processing</a></li>
<li class="chapter" data-level="4.7.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#chimera-detection"><i class="fa fa-check"></i><b>4.7.2</b> Chimera Detection</a></li>
<li class="chapter" data-level="4.7.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#otu-assignment-1"><i class="fa fa-check"></i><b>4.7.3</b> OTU Assignment</a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="measurement-process-1.html"><a href="measurement-process-1.html#downstream-analysis-1"><i class="fa fa-check"></i><b>4.8</b> Downstream analysis</a><ul>
<li class="chapter" data-level="4.8.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#taxonomic-classification"><i class="fa fa-check"></i><b>4.8.1</b> taxonomic classification</a></li>
<li class="chapter" data-level="4.8.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#differential-abundance-1"><i class="fa fa-check"></i><b>4.8.2</b> differential abundance</a></li>
<li class="chapter" data-level="4.8.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#ecological-diversity"><i class="fa fa-check"></i><b>4.8.3</b> ecological diversity</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="benchmarking-dataset.html"><a href="benchmarking-dataset.html"><i class="fa fa-check"></i><b>5</b> Benchmarking Dataset</a></li>
<li class="chapter" data-level="6" data-path="ecological-diversity-robustness.html"><a href="ecological-diversity-robustness.html"><i class="fa fa-check"></i><b>6</b> Ecological Diversity Robustness</a><ul>
<li class="chapter" data-level="6.1" data-path="ecological-diversity-robustness.html"><a href="ecological-diversity-robustness.html#overview"><i class="fa fa-check"></i><b>6.1</b> Overview</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="gold-standard-pipeline.html"><a href="gold-standard-pipeline.html"><i class="fa fa-check"></i><b>7</b> Gold Standard Pipeline</a></li>
<li class="chapter" data-level="8" data-path="assessment-framework.html"><a href="assessment-framework.html"><i class="fa fa-check"></i><b>8</b> Assessment Framework</a></li>
<li class="chapter" data-level="9" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>9</b> References</a></li>
<li class="divider"></li>
<li><a href="https://www.somewebsite.com" target="blank">Published with N8O</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Measurement Science of Metagenomics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="assessment-framework" class="section level1">
<h1><span class="header-section-number">8</span> Assessment Framework</h1>
<ul>
<li><p>Outline measurement process - fishbone diagram</p></li>
<li>Differentiate between biological variability and measurement error
<ul>
<li>How are these two sources of variability currently addressed in pipelines</li>
<li>Are there any known sources of biological variability not currently being addressed</li>
</ul></li>
<li>What is known about each source of variability?
<ul>
<li>uncertainty budget?<br />
</li>
<li>dark uncertianty - unaccounted for variability?</li>
</ul></li>
<li>Robustness testing
<ul>
<li>determine largest source of bias/ uncertianty in measurement process<br />
</li>
<li>Schmidt 2014 Repeatability and Reproducibility</li>
<li>Evaluation - no gold standard method, or known truth
<ul>
<li>Quantifying similarity in results
<ul>
<li>Schmidt et al 2014 Env Micro</li>
<li>Schloss et al 2016</li>
</ul></li>
<li>Method with similar assumptions</li>
<li>Methods with different assumption</li>
</ul></li>
</ul></li>
<li>Charaterize individual measurement process steps based on robustness testing/ sensitivity analysis
<ul>
<li>benchmarking datasets
<ul>
<li>environmental<br />
</li>
<li>in lab prepared material - combination of known samples<br />
</li>
<li>in lab prepared material - combination of unknown samples<br />
</li>
<li>in silico - various levels of simulation based on reference and environmental data availablility<br />
</li>
</ul></li>
<li>for each type of dataset summarize benefits and limitations to each</li>
</ul></li>
<li>Evaluation Framework
<ul>
<li>Requirements
<ul>
<li>Benchmarking dataset<br />
</li>
<li>Metric for evaluating performance
<ul>
<li>Assumption Based vs. Emerical Assessment
<ul>
<li>Schloss et al. paper using within and between cluster distances to benchmark clustering methods<br />
</li>
</ul></li>
<li>Thinking about what is being measured - biological question</li>
</ul></li>
</ul></li>
</ul></li>
<li>Comparison of methods
<ul>
<li>Benchmarking
<ul>
<li>Gold standard method - known correct answer
<ul>
<li>For new method if different how to determine if new method is better?<br />
</li>
</ul></li>
<li>Reference dataset - known truth<br />
</li>
</ul></li>
<li>Evaluation Methods
<ul>
<li>Benchmarking
<ul>
<li>Comparing results to known truth
<ul>
<li>limitation with biological applications no real ground truth.
<ul>
<li>For physical and chemical metrology a ground truth is determined through uncertainty analysis with traceability to the SI. Often requiring the use of two orthogonal methods and uncertainty budgets. Currently no methods for traceing sequence data to an SI.<br />
</li>
</ul></li>
</ul></li>
<li>Benchmarking datasets are generated from materials or <em>in silico</em> with a level of confidence regarding expected results.
<ul>
<li>Discussion of top down vs. bottom up uncertainty analysis for use in establishing truth with benchmarking datasets.<br />
</li>
</ul></li>
<li>Results from analysis of benchmarking datasets evaluated using performance metrics, e.g. truth tables for qualitative data.<br />
</li>
</ul></li>
<li>Benchmarking- optimization
<ul>
<li>evaluating method performance when subjected to changes in either quality of data (increased noise) or use of different methods or parameters (optimization) similar to sensitivity analysis but comparing results to a dataset with known truth.</li>
</ul></li>
</ul></li>
</ul></li>
</ul>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="gold-standard-pipeline.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page""><i class="fa fa-angle-right"></i></a>

<script src="libs/gitbook/js/app.js"></script>
<script src="libs/gitbook/js/lunr.js"></script>
<script src="libs/gitbook/js/plugin-search.js"></script>
<script src="libs/gitbook/js/plugin-sharing.js"></script>
<script src="libs/gitbook/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook/js/plugin-bookdown.js"></script>
<script src="libs/gitbook/js/jquery.highlight.js"></script>
<script>
require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/inst/examples/70-Assessment-Framework.Rmd",
"text": "Edit"
},
"download": null,
"toc": {
"collapse": false
},
"toolbar": {
"position": "fixed"
}
});
});
</script>


</body>

</html>
