<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Measurement Science of Metagenomics</title>
  <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
  <meta name="description" content="Measurement Science of Metagenomics">
  <meta name="generator" content="bookdown 0.1.3 and GitBook 2.6.7">

  <meta property="og:title" content="Measurement Science of Metagenomics" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Measurement Science of Metagenomics" />
  
  
  



<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  TeX: { equationNumbers: { autoNumber: "AMS" } }
});
</script>

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  

<link rel="next" href="s-rrna-gene.html">

<script src="libs/jquery/jquery.min.js"></script>
<link href="libs/gitbook/css/style.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>


  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Metrology of Marker-Gene Metagenomics </a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Metrology of Metagenomics</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#microbiome"><i class="fa fa-check"></i><b>1.1</b> Microbiome</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#measuring-microbiomes"><i class="fa fa-check"></i><b>1.2</b> Measuring Microbiomes</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#metrology"><i class="fa fa-check"></i><b>1.3</b> Metrology</a></li>
<li class="chapter" data-level="1.4" data-path="index.html"><a href="index.html#key-references"><i class="fa fa-check"></i><b>1.4</b> Key references</a><ul>
<li class="chapter" data-level="1.4.1" data-path="index.html"><a href="index.html#measurement-process"><i class="fa fa-check"></i><b>1.4.1</b> Measurement Process</a></li>
<li class="chapter" data-level="1.4.2" data-path="index.html"><a href="index.html#sample-processing-and-sequencing"><i class="fa fa-check"></i><b>1.4.2</b> Sample Processing and Sequencing</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="index.html"><a href="index.html#outline"><i class="fa fa-check"></i><b>1.5</b> Outline</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html"><i class="fa fa-check"></i><b>2</b> 16S rRNA Gene</a><ul>
<li class="chapter" data-level="2.1" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html#general-background-about-16s-rrna-gene"><i class="fa fa-check"></i><b>2.1</b> General background about 16S rRNA Gene</a></li>
<li class="chapter" data-level="2.2" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html#s-rrna-phylogenetics"><i class="fa fa-check"></i><b>2.2</b> 16S rRNA phylogenetics</a></li>
<li class="chapter" data-level="2.3" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html#s-rrna-databases"><i class="fa fa-check"></i><b>2.3</b> 16S rRNA Databases</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="process-characterization.html"><a href="process-characterization.html"><i class="fa fa-check"></i><b>3</b> Process Characterization</a></li>
<li class="chapter" data-level="4" data-path="measurement-process-1.html"><a href="measurement-process-1.html"><i class="fa fa-check"></i><b>4</b> Measurement Process</a><ul>
<li class="chapter" data-level="4.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-to-seq"><i class="fa fa-check"></i><b>4.1</b> Sample to Seq</a><ul>
<li class="chapter" data-level="4.1.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-collection"><i class="fa fa-check"></i><b>4.1.1</b> Sample Collection</a></li>
<li class="chapter" data-level="4.1.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-processing"><i class="fa fa-check"></i><b>4.1.2</b> Sample Processing</a></li>
<li class="chapter" data-level="4.1.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#s-rrna-pcr"><i class="fa fa-check"></i><b>4.1.3</b> 16S rRNA PCR</a></li>
<li class="chapter" data-level="4.1.4" data-path="measurement-process-1.html"><a href="measurement-process-1.html#library-preparation"><i class="fa fa-check"></i><b>4.1.4</b> Library Preparation</a></li>
<li class="chapter" data-level="4.1.5" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sequencing"><i class="fa fa-check"></i><b>4.1.5</b> Sequencing</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#seq-to-otu"><i class="fa fa-check"></i><b>4.2</b> Seq to OTU</a><ul>
<li class="chapter" data-level="4.2.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#preprocessing"><i class="fa fa-check"></i><b>4.2.1</b> Preprocessing</a></li>
<li class="chapter" data-level="4.2.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#otu-assignment"><i class="fa fa-check"></i><b>4.2.2</b> OTU Assignment</a></li>
<li class="chapter" data-level="4.2.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#taxonomic-assignment"><i class="fa fa-check"></i><b>4.2.3</b> Taxonomic Assignment</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#downstream-analysis"><i class="fa fa-check"></i><b>4.3</b> Downstream analysis</a><ul>
<li class="chapter" data-level="4.3.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#differential-abundance"><i class="fa fa-check"></i><b>4.3.1</b> Differential Abundance</a></li>
<li class="chapter" data-level="4.3.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#alpha-diversity"><i class="fa fa-check"></i><b>4.3.2</b> Alpha Diversity</a></li>
<li class="chapter" data-level="4.3.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#beta-diversity"><i class="fa fa-check"></i><b>4.3.3</b> Beta Diversity</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="measurement-process-1.html"><a href="measurement-process-1.html#step-by-step-protocols"><i class="fa fa-check"></i><b>4.4</b> Step-by-Step protocols</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html"><i class="fa fa-check"></i><b>5</b> 16S rRNA Analysis Method Assessment</a><ul>
<li class="chapter" data-level="5.1" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#sample-to-fastq"><i class="fa fa-check"></i><b>5.1</b> Sample to fastq</a></li>
<li class="chapter" data-level="5.2" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#seq-to-otu-1"><i class="fa fa-check"></i><b>5.2</b> Seq to OTU</a><ul>
<li class="chapter" data-level="5.2.1" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#pre-processing"><i class="fa fa-check"></i><b>5.2.1</b> Pre-processing</a></li>
<li class="chapter" data-level="5.2.2" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#chimera-detection"><i class="fa fa-check"></i><b>5.2.2</b> Chimera Detection</a></li>
<li class="chapter" data-level="5.2.3" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#otu-assignment-1"><i class="fa fa-check"></i><b>5.2.3</b> OTU Assignment</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#downstream-analysis-1"><i class="fa fa-check"></i><b>5.3</b> Downstream analysis</a><ul>
<li class="chapter" data-level="5.3.1" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#transformations"><i class="fa fa-check"></i><b>5.3.1</b> Transformations</a></li>
<li class="chapter" data-level="5.3.2" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#taxonomic-classification"><i class="fa fa-check"></i><b>5.3.2</b> taxonomic classification</a></li>
<li class="chapter" data-level="5.3.3" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#differential-abundance-1"><i class="fa fa-check"></i><b>5.3.3</b> Differential Abundance</a></li>
<li class="chapter" data-level="5.3.4" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#ecological-diversity"><i class="fa fa-check"></i><b>5.3.4</b> ecological diversity</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>6</b> References</a></li>
<li class="divider"></li>
<li><a href="https://www.somewebsite.com" target="blank">Published with N8O</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Measurement Science of Metagenomics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">Measurement Science of Metagenomics</h1>
</div>
<div id="metrology-of-metagenomics" class="section level1">
<h1><span class="header-section-number">1</span> Metrology of Metagenomics</h1>
<div id="microbiome" class="section level2">
<h2><span class="header-section-number">1.1</span> Microbiome</h2>
</div>
<div id="measuring-microbiomes" class="section level2">
<h2><span class="header-section-number">1.2</span> Measuring Microbiomes</h2>
<ul>
<li>whole metagenomic sequencing</li>
<li>marker-gene metagenomics</li>
<li>16S vs. WMS</li>
<li><span class="citation">Clooney et al. (<a href="#ref-Clooney2016">2016</a>)</span> compared WMS and 16S rRNA sequence data for environmental samples sequenced using two platforms MiSeq and PGM.
<ul>
<li>WMS clustering seperately form the 16S results <span class="citation">(Clooney et al. <a href="#ref-Clooney2016">2016</a>, fig. 1)</span>.</li>
</ul></li>
</ul>
</div>
<div id="metrology" class="section level2">
<h2><span class="header-section-number">1.3</span> Metrology</h2>
<ul>
<li>No not meteorlogy</li>
<li>Biological variability and measurement error
<ul>
<li>Measurement Error
<ul>
<li>Systematic - results in systematic bias</li>
<li>Random - increases uncertainty in results but provides an accurate measure of the true mean given sufficient sample size</li>
</ul></li>
<li>Biological Noise
<ul>
<li>Sources of variability due to biological complexity of the system e.g. difference in results based on 16S rRNA gene region</li>
</ul></li>
</ul></li>
<li>replicate, repeat, and reproduce
<ul>
<li>SI redefining the kilogram example???</li>
<li>Uncertainty budgets
<ul>
<li>top-down vs. bottom-up</li>
<li>analytical chemsitry text</li>
</ul></li>
<li>Orthogonal methods
<ul>
<li>Genome in a bottle example</li>
<li>Examples of orthogonal methods for 16S metagenomics:</li>
<li><span class="citation">Albertsen et al. (<a href="#ref-Albertsen2015">2015</a>)</span> 16S, metagenomics, metatranscriptomics, and FISH</li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="key-references" class="section level2">
<h2><span class="header-section-number">1.4</span> Key references</h2>
<div id="measurement-process" class="section level3">
<h3><span class="header-section-number">1.4.1</span> Measurement Process</h3>
<p><span class="citation">Goodrich et al. (<a href="#ref-Goodrich2014">2014</a>)</span></p>
</div>
<div id="sample-processing-and-sequencing" class="section level3">
<h3><span class="header-section-number">1.4.2</span> Sample Processing and Sequencing</h3>
<p><span class="citation">Brooks et al. (<a href="#ref-Brooks2015">2015</a>)</span><br />
<span class="citation">D’Amore et al. (<a href="#ref-Amore2016">2016</a>)</span></p>
</div>
</div>
<div id="outline" class="section level2">
<h2><span class="header-section-number">1.5</span> Outline</h2>
<ul>
<li>Background
<ul>
<li>Metagenomics and microbiome<br />
</li>
<li>Metrology<br />
</li>
</ul></li>
<li>Measurement Process
<ul>
<li>summary of individual steps<br />
</li>
<li>Downstream applications<br />
</li>
</ul></li>
<li>Sources of Error and Bias
<ul>
<li>Biological vs. Measurement Sources of error<br />
</li>
<li>error and bias for individual steps
<ul>
<li>subsections for biological and measurement?<br />
</li>
<li>mitigation strategies - wet lab and dry lab<br />
</li>
</ul></li>
</ul></li>
<li>Measurement Assessment
<ul>
<li>Overview with historical context<br />
</li>
<li>Assessment methods for individual steps in measurement process from a data analysis perspective</li>
</ul></li>
</ul>
<!--

Remember each Rmd file contains one and only one chapter.

## Book Outline

## Metagenomics
* Microbiome
* Measuring Microbiomes
    - whole metagenomic sequencing
    - marker-gene metagenomics
        + 16S rRNA gene

## Marker-gene metagenomics measurement process
* Measurement Process overview
    - Sample collection
    - Sample Processing
    - Sequencing
    - Data analysis
        + fastq to OTU
        + Downstream analysis
            * differential abundance
            * alpha diversity
            * beta diversity

## Biological variability and measurement error
* Measurement Error
    - Systematic - results in systematic bias
    - Random - increases uncertainty in results but provides an accurate measure of the true mean given sufficient sample size
* Biological Noise
    - Sources of variability due to biological complexity of the system e.g. difference in results based on 16S rRNA gene region
* replicate, repeat, and reproduce
    - Uncertainty budgets
        + top-down vs. bottom-up
    - Orthogonal methods

## Sources of variability and error for marker-gene metagenomics
* fishbone diagram
    - Sample collection
    - Sample Processing
        + DNA extraction
        + PCR/ indexing
        + Normalization/ Pooling
    - Sequencing
        + sequencing reaction
        + imaging
        + base calling
    - Data analysis
        + Pre-processing (wheat from the chaf)
            * quality filter/ read trimming
            * chimera detection
            * clustering
        + Taxonomic classification
        + Downstream analysis
            * differential abundance
            * alpha diversity
            * beta diversity
* cause effect diagram
    - pre-processing, normalization/ transformation methods used to address measurement error

## Characterizing Measurement Process
* Tools
    - Validation
        + Fit for purpose - comparison to gold standard method or reference material
            * no gold standard methods - only community accepted defactor standards
    - Benchmarking
        + Comparison to known truth
        + Evaluating results based on expectation
    - Robustness testing
        + Impact of procedure or step on results
* Characterization of Data Analysis Processing
    - Wet lab vs. dry lab
    - Validation
        + Currently not applicable ... -> most robust method
    - Benchmarking
        + Observed results vs. Truth
            * Dataset with known true value - can be obtained using
                - gold standard method
                - mock communities: in vitro or in silico
                    + limitations and assumptions
                - Mixtures
                    + Provides a degree of known truth
            * Performance metrics
                - Quantitative
                    + Variability - standard deviation/ variance
                    + Bias - mean squared error??? (true value)
        + Observed results vs. assumptions
            * No general dataset requirements
            * Performance metrics
                - evaluate results based on method assumption, e.g. Schloss clustering distance
    - Robustness Testing
        + Used to evaluate impact of different methods on results
        + Comparison of orthogonal methods
            * For fundamentally different methods (different assumptions and heuristics)
                - Can evaluate precision (variability), not necessarily accuracy but less susceptible to bias than is methods are based on the same assumptions or heuristics
        + Used to evaluate how different steps in the process affect the results

## Microbiome Mixture Benchmarking Dataset
* Mixture Study

## Robustness Testing Ecological Diversity
* Evaluating alpha and beta diversity metrics
* Factors
    - preprocessing methods
    - data characteristics
        + sequencing error rates
        + primer region
        + read length
* OTU table
    - depth/ coverage - sparsity
    - when does the diversity metric break?

## Guideance on developing an assessment framework
* Outline measurement process - fishbone diagram
* Differentiate between biological variability and measurement error
    - How are these two sources of variability currently addressed in pipelines
    - Are there any known sources of biological variability not currently being addressed
* What is known about each source of variability?
    - uncertainty budget?
    - dark uncertianty - unaccounted for variability?
* Robustness testing
    - determine largest source of bias/ uncertianty in measurement process
* Charaterize individual measurement process steps based on robustness testing/ sensitivity analysis
    - benchmarking datasets
        + environmental
        + in lab prepared material - combination of known samples
        + in lab prepared material - combination of unknown samples
        + in silico - various levels of simulation based on reference and environmental data availablility
    - for each type of dataset summarize benefits and limitations to each

## Proposed Gold Standard Analysis Method
* Trim primers - dereplicate
* Chimera check? - multiple methods orthogonal methods?, annotate don't filter
* BLAST 100% - all perfect matches
* Unmatched
    - Multiple sequence alignment
        + Filter sequences with low alignment scores
    - phylogenetic placement
* Generate OTUs for different branch distances
* Keep unique singletons - based on distance from non-singleton clusters, vary similarity based on cluster size, likelihood model for whether sequencing error or biological mutation.
    - Rare orgs highly similar to large OTU less likely ecologically different?
* Alternative taxa classification
    - Multiple methods
        + pairwise alignment
        + kmer
        + phylogenetic placement
        + other...
    - Multiple reference databases
        + evaluate individual sequences based on confidence - presence in multiple databases, infernal and rnammer alignment score?
            * supporting sequence data - whole genome, sanger, 16S PCR from isolate
            * likelihood of being a chimera
        + Compare other methods to results generated using gold standard pipeline
            * performance metrics
* Copy number correction
    - based on phylogenetic placement
-->

</div>
</div>
<h3> References</h3>
<div id="refs" class="references">
<div id="ref-Clooney2016">
<p>Clooney, Adam G, Fiona Fouhy, Roy D Sleator, Aisling O’ Driscoll, Catherine Stanton, Paul D Cotter, and Marcus J Claesson. 2016. “Comparing Apples and Oranges?: Next Generation Sequencing and Its Impact on Microbiome Analysis.” <em>PloS One</em> 11 (2): e0148028. doi:<a href="https://doi.org/10.1371/journal.pone.0148028">10.1371/journal.pone.0148028</a>.</p>
</div>
<div id="ref-Albertsen2015">
<p>Albertsen, Mads, Søren M Karst, Anja S Ziegler, Rasmus H Kirkegaard, and Per H Nielsen. 2015. “Back to Basics - The Influence of DNA Extraction and Primer Choice on Phylogenetic Analysis of Activated Sludge Communities.” <em>PloS One</em> 10 (7). Public Library of Science: e0132783. doi:<a href="https://doi.org/10.1371/journal.pone.0132783">10.1371/journal.pone.0132783</a>.</p>
</div>
<div id="ref-Goodrich2014">
<p>Goodrich, Julia K., Sara C. Di Rienzi, Angela C. Poole, Omry Koren, William A. Walters, J. Gregory Caporaso, Rob Knight, and Ruth E. Ley. 2014. “Conducting a Microbiome Study.” <em>Cell</em> 158 (2). Elsevier Inc.: 250–62. doi:<a href="https://doi.org/10.1016/j.cell.2014.06.037">10.1016/j.cell.2014.06.037</a>.</p>
</div>
<div id="ref-Brooks2015">
<p>Brooks, J Paul, David J Edwards, Michael D Harwich, Maria C Rivera, Jennifer M Fettweis, Myrna G Serrano, Robert A Reris, et al. 2015. “The truth about metagenomics: quantifying and counteracting bias in 16S rRNA studies.” <em>BMC Microbiology</em> 15 (1): 66. doi:<a href="https://doi.org/10.1186/s12866-015-0351-6">10.1186/s12866-015-0351-6</a>.</p>
</div>
<div id="ref-Amore2016">
<p>D’Amore, Rosalinda, Umer Zeeshan Ijaz, Melanie Schirmer, John G Kenny, Richard Gregory, Alistair C Darby, Christopher Quince, et al. 2016. “A comprehensive benchmarking study of protocols and sequencing platforms for 16S rRNA community profiling.” <em>BMC Genomics</em> 17. BMC Genomics: 1–40. doi:<a href="https://doi.org/10.1186/s12864-015-2194-9">10.1186/s12864-015-2194-9</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="s-rrna-gene.html" class="navigation navigation-next navigation-unique" aria-label="Next page""><i class="fa fa-angle-right"></i></a>

<script src="libs/gitbook/js/app.min.js"></script>
<script src="libs/gitbook/js/lunr.js"></script>
<script src="libs/gitbook/js/plugin-search.js"></script>
<script src="libs/gitbook/js/plugin-sharing.js"></script>
<script src="libs/gitbook/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook/js/plugin-bookdown.js"></script>
<script src="libs/gitbook/js/jquery.highlight.js"></script>
<script>
require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/nate-d-olson/mgmgnotes/index.Rmd",
"text": "Edit"
},
"download": null,
"toc": {
"collapse": false
},
"toolbar": {
"position": "fixed"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
