<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Measurement Science of Metagenomics</title>
  <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
  <meta name="description" content="Measurement Science of Metagenomics">
  <meta name="generator" content="bookdown 0.0.52 and GitBook 2.6.7">



<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  TeX: { equationNumbers: { autoNumber: "AMS" } }
});
</script>
  <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="process-characterization.html">
<link rel="next" href="s-rrna-analysis-method-assessment.html">

<link href="libs/gitbook/css/style.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>


  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Metrology of Marker-Gene Metagenomics </a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Metrology of Metagenomics</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#microbiome"><i class="fa fa-check"></i><b>1.1</b> Microbiome</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#measuring-microbiomes"><i class="fa fa-check"></i><b>1.2</b> Measuring Microbiomes</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#metrology"><i class="fa fa-check"></i><b>1.3</b> Metrology</a></li>
<li class="chapter" data-level="1.4" data-path="index.html"><a href="index.html#key-references"><i class="fa fa-check"></i><b>1.4</b> Key references</a><ul>
<li class="chapter" data-level="1.4.1" data-path="index.html"><a href="index.html#measurement-process"><i class="fa fa-check"></i><b>1.4.1</b> Measurement Process</a></li>
<li class="chapter" data-level="1.4.2" data-path="index.html"><a href="index.html#sample-processing-and-sequencing"><i class="fa fa-check"></i><b>1.4.2</b> Sample Processing and Sequencing</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html"><i class="fa fa-check"></i><b>2</b> 16S rRNA Gene</a><ul>
<li class="chapter" data-level="2.1" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html#general-background-about-16s-rrna-gene"><i class="fa fa-check"></i><b>2.1</b> General background about 16S rRNA Gene</a></li>
<li class="chapter" data-level="2.2" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html#s-rrna-phylogenetics"><i class="fa fa-check"></i><b>2.2</b> 16S rRNA phylogenetics</a></li>
<li class="chapter" data-level="2.3" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html#s-rrna-databases"><i class="fa fa-check"></i><b>2.3</b> 16S rRNA Databases</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="process-characterization.html"><a href="process-characterization.html"><i class="fa fa-check"></i><b>3</b> Process Characterization</a></li>
<li class="chapter" data-level="4" data-path="measurement-process-1.html"><a href="measurement-process-1.html"><i class="fa fa-check"></i><b>4</b> Measurement Process</a><ul>
<li class="chapter" data-level="4.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-to-seq"><i class="fa fa-check"></i><b>4.1</b> Sample to Seq</a><ul>
<li class="chapter" data-level="4.1.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-collection"><i class="fa fa-check"></i><b>4.1.1</b> Sample Collection</a></li>
<li class="chapter" data-level="4.1.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-processing"><i class="fa fa-check"></i><b>4.1.2</b> Sample Processing</a></li>
<li class="chapter" data-level="4.1.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#s-rrna-pcr"><i class="fa fa-check"></i><b>4.1.3</b> 16S rRNA PCR</a></li>
<li class="chapter" data-level="4.1.4" data-path="measurement-process-1.html"><a href="measurement-process-1.html#library-preparation"><i class="fa fa-check"></i><b>4.1.4</b> Library Preparation</a></li>
<li class="chapter" data-level="4.1.5" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sequencing"><i class="fa fa-check"></i><b>4.1.5</b> Sequencing</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#seq-to-otu"><i class="fa fa-check"></i><b>4.2</b> Seq to OTU</a><ul>
<li class="chapter" data-level="4.2.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#quality-filter-read-trimming"><i class="fa fa-check"></i><b>4.2.1</b> quality filter/ read trimming</a></li>
<li class="chapter" data-level="4.2.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#otu-assignment"><i class="fa fa-check"></i><b>4.2.2</b> OTU Assignment</a></li>
<li class="chapter" data-level="4.2.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#taxonomic-assignment"><i class="fa fa-check"></i><b>4.2.3</b> Taxonomic Assignment</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#downstream-analysis"><i class="fa fa-check"></i><b>4.3</b> Downstream analysis</a><ul>
<li class="chapter" data-level="4.3.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#differential-abundance"><i class="fa fa-check"></i><b>4.3.1</b> Differential Abundance</a></li>
<li class="chapter" data-level="4.3.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#alpha-diversity"><i class="fa fa-check"></i><b>4.3.2</b> Alpha Diversity</a></li>
<li class="chapter" data-level="4.3.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#beta-diversity"><i class="fa fa-check"></i><b>4.3.3</b> Beta Diversity</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="measurement-process-1.html"><a href="measurement-process-1.html#step-by-step-protocols"><i class="fa fa-check"></i><b>4.4</b> Step-by-Step protocols</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html"><i class="fa fa-check"></i><b>5</b> 16S rRNA Analysis Method Assessment</a><ul>
<li class="chapter" data-level="5.1" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#sample-to-fastq"><i class="fa fa-check"></i><b>5.1</b> Sample to fastq</a></li>
<li class="chapter" data-level="5.2" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#seq-to-otu-1"><i class="fa fa-check"></i><b>5.2</b> Seq to OTU</a><ul>
<li class="chapter" data-level="5.2.1" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#pre-processing"><i class="fa fa-check"></i><b>5.2.1</b> Pre-processing</a></li>
<li class="chapter" data-level="5.2.2" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#chimera-detection"><i class="fa fa-check"></i><b>5.2.2</b> Chimera Detection</a></li>
<li class="chapter" data-level="5.2.3" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#otu-assignment-1"><i class="fa fa-check"></i><b>5.2.3</b> OTU Assignment</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#downstream-analysis-1"><i class="fa fa-check"></i><b>5.3</b> Downstream analysis</a><ul>
<li class="chapter" data-level="5.3.1" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#taxonomic-classification"><i class="fa fa-check"></i><b>5.3.1</b> taxonomic classification</a></li>
<li class="chapter" data-level="5.3.2" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#differential-abundance-1"><i class="fa fa-check"></i><b>5.3.2</b> differential abundance</a></li>
<li class="chapter" data-level="5.3.3" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#ecological-diversity"><i class="fa fa-check"></i><b>5.3.3</b> ecological diversity</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="benchmarking-dataset.html"><a href="benchmarking-dataset.html"><i class="fa fa-check"></i><b>6</b> Benchmarking Dataset</a></li>
<li class="chapter" data-level="7" data-path="ecological-diversity-robustness.html"><a href="ecological-diversity-robustness.html"><i class="fa fa-check"></i><b>7</b> Ecological Diversity Robustness</a><ul>
<li class="chapter" data-level="7.1" data-path="ecological-diversity-robustness.html"><a href="ecological-diversity-robustness.html#overview"><i class="fa fa-check"></i><b>7.1</b> Overview</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="gold-standard-pipeline.html"><a href="gold-standard-pipeline.html"><i class="fa fa-check"></i><b>8</b> Gold Standard Pipeline</a></li>
<li class="chapter" data-level="9" data-path="assessment-framework.html"><a href="assessment-framework.html"><i class="fa fa-check"></i><b>9</b> Assessment Framework</a></li>
<li class="chapter" data-level="10" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>10</b> References</a></li>
<li class="divider"></li>
<li><a href="https://www.somewebsite.com" target="blank">Published with N8O</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Measurement Science of Metagenomics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="measurement-process-1" class="section level1">
<h1><span class="header-section-number">4</span> Measurement Process</h1>
<p>Overview of the 16S rRNA marker gene metagenomics measurement process including the sources of error and bias introduced at different step in the process along with different mitigation strategies that have been developed to address each of the issues. See <span class="citation">Goodrich et al. (<a href="references.html#ref-Goodrich2014">2014</a>)</span> for a review of 16S rRNA marker gene metagenomics measurement process and general recomendations for conducting a microbiome study.</p>
<p><span class="citation">J. Zhou et al. (<a href="references.html#ref-Zhou2011">2011</a>)</span> Used biological and technical replicates with a <em>Shewanella oneidenis MR-1</em> to assess the reproducibility and quanitative accuracy of 16S sequencing, using 454 pyroseuqencing and targeting the V45 region. Results demonstrated a lack of reporducibility and quantiative accuracy when comparing the <span class="math inline">\(B\)</span> diversity and relative abundance between replicates.</p>
<p>Diagram of measurement process <a href="measurement-process-1.html#fig:fishbone">4.1</a>.</p>
<pre><code>## Package &#39;qcc&#39;, version 2.6</code></pre>
<pre><code>## Type &#39;citation(&quot;qcc&quot;)&#39; for citing this R package in publications.</code></pre>
<div class="figure"><span id="fig:fishbone"></span>
<img src="_main_files/figure-html/fishbone-1.png" alt="Fishbone diagram of the 16S rRNA metagenomic sequencing measurement process."  />
<p class="caption">
Figure 4.1: Fishbone diagram of the 16S rRNA metagenomic sequencing measurement process.
</p>
</div>
<p>Review of what is currently known about the sources of error and bias in the 16S metagenomics measurement process. Cause-effect diagrams are commonly used in process quality control to show the relationship between steps in the measurement process and sources of error or bias <a href="measurement-process-1.html#fig:cause-effect">4.2</a>.<br />
Potentially may want to revise to focus on sources of error, e.g. cell type bias, preferential amplification, sequence errors, chimeras .</p>
<div class="figure"><span id="fig:cause-effect"></span>
<img src="_main_files/figure-html/cause-effect-1.png" alt="Causes (branch labels) of bias and variability indicated for different sources of error (branch text)."  />
<p class="caption">
Figure 4.2: Causes (branch labels) of bias and variability indicated for different sources of error (branch text).
</p>
</div>
<div id="sample-to-seq" class="section level2">
<h2><span class="header-section-number">4.1</span> Sample to Seq</h2>
<p>The first part of the measurement process is generating sequencing data from an environmental sample.</p>
<p>The part consists of five main steps:</p>
<ol style="list-style-type: decimal">
<li>Sample Collection<br />
</li>
<li>Sample Processing<br />
</li>
<li>16S rRNA PCR<br />
</li>
<li>Library Prep<br />
</li>
<li>Sequencing</li>
</ol>
<div id="sample-collection" class="section level3">
<h3><span class="header-section-number">4.1.1</span> Sample Collection</h3>
<ul>
<li>Collection of sample that is representative of population being studied
<ul>
<li>Kitty Biome study as an example of sample heterogeneity<br />
</li>
</ul></li>
<li>Impact of sample storage
<ul>
<li>Fecal storage study<br />
</li>
<li>Old coral study?</li>
</ul></li>
</ul>
</div>
<div id="sample-processing" class="section level3">
<h3><span class="header-section-number">4.1.2</span> Sample Processing</h3>
<div id="dna-extraction" class="section level4">
<h4><span class="header-section-number">4.1.2.1</span> DNA extraction</h4>
<p>Considerations:</p>
<ul>
<li>Cell type specific extraction biases<br />
</li>
<li>DNA concentration requirements for library prep
<ul>
<li>Issues with low concentration samples - skin microbiome<br />
</li>
</ul></li>
<li>Extra cellular DNA</li>
<li>PCR inhibitiors
<ul>
<li>Old PCR inhibitor paper.<br />
</li>
<li>Any studies showing inhibitors inducing biases in metagenomic studies?
<ul>
<li>Potentially sample specific biases but do not know of any within sample biases</li>
</ul></li>
</ul></li>
</ul>
<p><span class="citation">Albertsen et al. (<a href="references.html#ref-Albertsen2015">2015</a>)</span> evalauted the impact of different DNA extraction methods for activated slude communities. Bead beating both time and intensity impacted the yield and size distribution of the extracted DNA as well as the observed abundance of different bacteria at both the phylum level and class for Proteobacteria.</p>
</div>
</div>
<div id="s-rrna-pcr" class="section level3">
<h3><span class="header-section-number">4.1.3</span> 16S rRNA PCR</h3>
<div id="template-concentration" class="section level4">
<h4><span class="header-section-number">4.1.3.1</span> Template concentration</h4>
<ul>
<li>This is related to low concentration samples or samples with poor DNA extraction efficiencies.</li>
</ul>
<p><span class="citation">Kennedy et al. (<a href="references.html#ref-Kennedy2014">2014</a>)</span> showed that the profile of replicate samples for different template concentration was more variable then when the imput concentration is the same. Pooling replicate PCR reactions decreased the within sample variablility.</p>
<p><span class="citation">D’Amore et al. (<a href="references.html#ref-Amore2016">2016</a>)</span> increasing the template concentration from 1 ng to 10 ng for MiSeq lead to higher percentages of chimeric reads 0.08 compared to 0.2% (p = 0.20).</p>
</div>
<div id="target-variable-region" class="section level4">
<h4><span class="header-section-number">4.1.3.2</span> Target variable region</h4>
<p><span class="citation">Clooney et al. (<a href="references.html#ref-Clooney2016">2016</a>)</span> Compared sequencing data between platforms and variable regions. Based on the relative abundance of genera present in 20% of the datasets (including whole metagenome sequencing (WMS)) the V12 and V45 primer regions, PGM sequencing data was less impacted than MiSeq to different variable regions.</p>
<p><span class="citation">Kozich et al. (<a href="references.html#ref-Kozich2013">2013</a>)</span> compared the results for three different variable regions sequenced using the MiSeq platform. The three regions, V34, V4, and V45 represented different length (250 bp, 430 bp, and 375 bp) though overlapping target regions. The observed error rate varied between run as well as variable regions resulting in differences in predicted OTUs. V45 had the highest predicted OTUs for all samples and replicate sequencing runs, followed by V34 and V4. The number of observed OTUs was higher for all datasets than expected for the mock community datasets.</p>
</div>
<div id="pcr-primers-optional-barcoded-primers" class="section level4">
<h4><span class="header-section-number">4.1.3.3</span> PCR primers (optional barcoded primers)</h4>
<p><span class="citation">Albertsen et al. (<a href="references.html#ref-Albertsen2015">2015</a>)</span> performed both an <em>in silico</em> and <em>in vitro</em> comparison of primer regions as well as comparing the relative abundance of different Phyla to metagenomic and metatranscriptomic datasets. The author’s concluded by recomending V13 for activated sludge samples based on what is known about the community composition of the system.</p>
</div>
<div id="pcr-reagents" class="section level4">
<h4><span class="header-section-number">4.1.3.4</span> PCR reagents</h4>
<ul>
<li>Polymerase - high fidelity<br />
</li>
<li>Reagent contaminats</li>
</ul>
</div>
<div id="pcr-cycling-conditions" class="section level4">
<h4><span class="header-section-number">4.1.3.5</span> PCR cycling conditions</h4>
</div>
<div id="number-of-cycles" class="section level4">
<h4><span class="header-section-number">4.1.3.6</span> number of cycles</h4>
<p>Using a mock community of 68 organisms <span class="citation">D’Amore et al. (<a href="references.html#ref-Amore2016">2016</a>)</span> showed that increasing the number of PCR cycles resulted in increased sequencing error rates (0.58% vs. 0.64% p = 0.11) and chimeras (15 cycles 0.00% and 25 cycles 0.66%, p = 0.0245).</p>
<p><span class="citation">Albertsen et al. (<a href="references.html#ref-Albertsen2015">2015</a>)</span> evaluated the amount of template and cycle number but only observed “small changes”. Decreasing the annealing temperatures of 58C and 52C resulted in increased abundance of prevously low abundance OTU with no impact on high abundance OTUs.</p>
<p><span class="citation">Yu et al. (<a href="references.html#ref-Yu2015">2015</a>)</span> compared standard and nest PCR (barcodes used in the second round), showed that the number of cycles and use of separate PCR and indexing reactions influenced the number of observed OTUs and <span class="math inline">\(\B\)</span> diversity.</p>
</div>
<div id="sequence-copy-abundance" class="section level4">
<h4><span class="header-section-number">4.1.3.7</span> sequence copy abundance</h4>
</div>
<div id="chimeras" class="section level4">
<h4><span class="header-section-number">4.1.3.8</span> chimeras</h4>
</div>
</div>
<div id="library-preparation" class="section level3">
<h3><span class="header-section-number">4.1.4</span> Library Preparation</h3>
<p><span class="citation">Kennedy et al. (<a href="references.html#ref-Kennedy2014">2014</a>)</span> found that variability due to PCR (particularly template concentration) resulted in less variability for within sample results than sample preparation (library preparation) and inter-lane variability. Barcoded 16S PCR primers were used in the study therefore the impact of barcodes and variability of 16S PCR was confounded.</p>
<p><span class="citation">Schirmer et al. (<a href="references.html#ref-Schirmer2015">2015</a>)</span> sequenced a mock community of 59 different strains to evaluate the sequencing error rate for different sample processing methods. Though only fully characterizing a subset of a the results.</p>
<div id="barcode-indexing-optional" class="section level4">
<h4><span class="header-section-number">4.1.4.1</span> Barcode/ indexing (optional)</h4>
<ul>
<li>Do not know of any barcode specific biases</li>
</ul>
</div>
<div id="normalization-and-pooling" class="section level4">
<h4><span class="header-section-number">4.1.4.2</span> Normalization and Pooling</h4>
<ul>
<li>Methods
<ul>
<li>DNA quantification<br />
</li>
<li>Bead based assays<br />
</li>
</ul></li>
<li>Associated Biases
<ul>
<li>Uneven sequencing depth - impact of coverage</li>
</ul></li>
</ul>
</div>
</div>
<div id="sequencing" class="section level3">
<h3><span class="header-section-number">4.1.5</span> Sequencing</h3>
<ul>
<li>Sequencing platform<br />
</li>
<li><span class="citation">Clooney et al. (<a href="references.html#ref-Clooney2016">2016</a>)</span> compared 16S rRNA sequence data for environmental samples sequenced using two platforms MiSeq and PGM.
<ul>
<li>Comparing results between stool samples from six individuals based on the relative abundance of genera present in 20% of the datasets (including whole metagenome sequencing (WMS)) the datasets clustered together based on sequencing methods (platform and primers) rather than individual, with WMS clustering seperately form the 16S results <span class="citation">(Clooney et al. <a href="references.html#ref-Clooney2016">2016</a>, fig. 1)</span>.<br />
</li>
</ul></li>
<li>Read length<br />
</li>
<li>Sequencing chemistry</li>
</ul>
</div>
</div>
<div id="seq-to-otu" class="section level2">
<h2><span class="header-section-number">4.2</span> Seq to OTU</h2>
<p>Convering raw sequence data to a OTU table with taxonomic annotations for use in downstream applications.</p>
<div id="quality-filter-read-trimming" class="section level3">
<h3><span class="header-section-number">4.2.1</span> quality filter/ read trimming</h3>
<ul>
<li>Filtering low quality reads - short reads with high error rates.<br />
</li>
<li>Trimming reads - removing ends of reads based on base quality score as well as removal of adapter sequences that were added during sample processing.</li>
<li>Merging paired ends</li>
<li>Merging forward and reverse reads into single contigs</li>
<li>chimera detection</li>
<li>detection and removal of chimeric reads, artifact of the PCR amplification process.</li>
</ul>
</div>
<div id="otu-assignment" class="section level3">
<h3><span class="header-section-number">4.2.2</span> OTU Assignment</h3>
<p>OTU assignment most commonly by clustering is used to reduce the number of spurious observations, sequences are grouped into OTUs.</p>
<ul>
<li>Different approaches to clustering
<ul>
<li>distance based clustering - most commonly used
<ul>
<li>Distance based clustering is a NP hard problem (i.e. computationaly intense), therefore a number of heuristics are used to make the method computationally tractable for large datasets.</li>
<li>Threshold are used to define the similarity of sequences within a cluster, 99% and 97% are commonly used.
<ul>
<li>Some biological and sequencing error basis for threshold values, (e.g. 99% species level, 97% genus level, 1% error rate). Taxonomic variability in 16S rRNA sequence diversity (Pai?) and sequencing error <span class="citation">(Huse et al. <a href="references.html#ref-Huse2010">2010</a>, <span class="citation">Schirmer et al. (<a href="references.html#ref-Schirmer2015">2015</a>)</span>, <span class="citation">D’Amore et al. (<a href="references.html#ref-Amore2016">2016</a>)</span>)</span> challenges the validity of these values.</li>
</ul></li>
<li>Different distance based thresholds.
<ul>
<li>average linkage<br />
</li>
<li>single linkage<br />
</li>
<li>complete linkage<br />
</li>
</ul></li>
</ul></li>
<li>Other clustering/ OTU assignment methods include:
<ul>
<li>distribution based<br />
</li>
<li>oligotyping<br />
</li>
<li>tree based clustering <span class="citation">(D. Wu, Doroud, and Eisen <a href="references.html#ref-Wu2013">2013</a>)</span></li>
</ul></li>
<li>De novo vs Reference based clustering
<ul>
<li>Sequences can be clustered with a set of reference cluster centers or without (<em>de novo</em>).</li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="taxonomic-assignment" class="section level3">
<h3><span class="header-section-number">4.2.3</span> Taxonomic Assignment</h3>
<p>Assign clusters a taxonomic name * <span class="citation">(“Comparison of two next-generation sequencing technologies for resolving highly complex microbiota composition using tandem variable 16S rRNA gene regions.” <a href="references.html#ref-Claesson2010">2010</a>)</span> <em>in silico</em> analysis of the taxonomic resolution for different variable regions and read lengths (454 and MISeq 150, 100, 75, and 50 bp paired end reads) and primers.</p>
</div>
</div>
<div id="downstream-analysis" class="section level2">
<h2><span class="header-section-number">4.3</span> Downstream analysis</h2>
<div id="differential-abundance" class="section level3">
<h3><span class="header-section-number">4.3.1</span> Differential Abundance</h3>
<ul>
<li>Differences in taxa or OTU abundance between experimental factors, e.g. treatment vs. control.</li>
<li><span class="citation">Jonsson et al. (<a href="references.html#ref-Jonsson2016">2016</a>)</span> Simulation based approach to evaluating differential abundance methods. <strong>NEED TO REVIEW</strong></li>
</ul>
</div>
<div id="alpha-diversity" class="section level3">
<h3><span class="header-section-number">4.3.2</span> Alpha Diversity</h3>
<ul>
<li>Measure of biological or ecological diversity within a sample.<br />
</li>
<li>Alpha diversity as a continuum related to the metric dependence on richness and evenness <span class="citation">(Chao et al. <a href="references.html#ref-Chao2014">2014</a>)</span></li>
<li>Four primary categories richness, evenness, abundance weighted, and dissimilarity <span class="citation">(Ricotta <a href="references.html#ref-Ricotta2007">2007</a>)</span>
<ul>
<li>Richness
<ul>
<li>Hill numbers, rarefaction, and richness estimation <span class="citation">(Chao et al. <a href="references.html#ref-Chao2014">2014</a>)</span></li>
</ul></li>
<li>Evenness
<ul>
<li>Criteria for evaluating evenness metrics <span class="citation">(Tuomisto <a href="references.html#ref-Tuomisto2012">2012</a>)</span></li>
<li>See <span class="citation">Tuomisto (<a href="references.html#ref-Tuomisto2012">2012</a>)</span> Table 1 for summary of evenness metric including interpretation</li>
</ul></li>
<li>Abundance weighted</li>
<li>Dissimilarity</li>
</ul></li>
<li>Parametric vs. Non-parametric diversity metrics for 16S rRNA sequence analysis <span class="citation">(Bunge, Willis, and Walsh <a href="references.html#ref-Bunge2014">2014</a>)</span></li>
<li>Development of alpha diversity richness metric for microbial diversity.
<ul>
<li>Chao1 a traditional diversity metric estimates total diversity using the number of singletons, OTUs representing only a single sequence.
<ul>
<li>Issue with singleton based diversity metrics for microbial diversity is that for incresed sequencing depth the number of singletons with continue to increase due to sequencing errors, leading to artifically inflated diversity estimates.</li>
</ul></li>
<li><span class="citation">Chiu and Chao (<a href="references.html#ref-Chiu2015">2015</a>)</span> developed a modified version of the Chao1 estimator that is more robust to singletons bue to sequencing errors.</li>
<li>This method estimates diversity based on the number of doubletons to tentons.<br />
</li>
<li>Incorporates Hill numbers as well.</li>
</ul></li>
</ul>
</div>
<div id="beta-diversity" class="section level3">
<h3><span class="header-section-number">4.3.3</span> Beta Diversity</h3>
</div>
</div>
<div id="step-by-step-protocols" class="section level2">
<h2><span class="header-section-number">4.4</span> Step-by-Step protocols</h2>
<ul>
<li><span class="citation">Albertsen et al. (<a href="references.html#ref-Albertsen2015">2015</a>)</span> Activated Sludge <a href="http://www.midasfieldguide.org/" class="uri">http://www.midasfieldguide.org/</a></li>
</ul>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="process-characterization.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="s-rrna-analysis-method-assessment.html" class="navigation navigation-next " aria-label="Next page""><i class="fa fa-angle-right"></i></a>

<script src="libs/gitbook/js/app.js"></script>
<script src="libs/gitbook/js/lunr.js"></script>
<script src="libs/gitbook/js/plugin-search.js"></script>
<script src="libs/gitbook/js/plugin-sharing.js"></script>
<script src="libs/gitbook/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook/js/plugin-bookdown.js"></script>
<script src="libs/gitbook/js/jquery.highlight.js"></script>
<script>
require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/inst/examples/20-Measurement-Process.Rmd",
"text": "Edit"
},
"download": null,
"toc": {
"collapse": false
},
"toolbar": {
"position": "fixed"
}
});
});
</script>


</body>

</html>
