# Proposal  
## Specific Aims Intro  

## General Background 
* Overview of marker gene sequencing methods  
* Fishbone diagram or schematic show sources of variability and error  

## Aim 1

### Title
Generate a sequence dataset for use in evaluating differential abundance marker-gene metagenomic analysis methods.  

### Summary
Sources of variability, biases, and error associated with 16S sequencing methods and analysis are not well characterized. 
Using mixtures of genomic DNA extracted from stool samples collected from volunteers before and after exposure to pathogenic E. coli we will generate a dataset that will allow us to characterize the contribution of each step in the measurement process to the overall variability in the sequencing method. 
The impact of the study is two fold; sequence generated will provide the community with a dataset benchmarking sequence analysis pipelines. 
The experimental design employed in the study can be used as a template for future studies evaluating the performance of other high-throughput sequencing methods.  

### Background and Significance

#### Background
* use of mixtures  
* datasets used in benchmarking  

#### Significance
* rich metadata, for optimal reuse  
* demonstration use of dataset to evaluate differential abundance methods  

### Approach

#### Sub Aims
1. We will generate a dataset for use in evaluating accuracy of relative abundance estimates.  
1. We will use the generated dataset to evaluate sources of variability.  
1. We will develop an R package for the dataset with extensive metadata and tools for working with the benchmarking dataset.   


#### Overview
* Generate a series of two sample titrations.  
* Assuming no variability or bias in the measurement process, the OTU counts for the titrations is linear. Any variability from the linear model represents a source of variability or error in the measurement process.  
* Through the use technical replicates and quality control measurements assess the sources of variability at each step in the measurement process.  
* Generate a curated set of OTUs for use in evaluating sample processing variability  
* Compare to linearity for standard pipelines  
* Potentially correct for sample processing variability  

#### Approach Details  
* Experimental design  
    -  5 biological replicates  
    -  2 treatment levels - pre and post exposure to E. coli  
    -  9 mixtures - unmixed pre and post, 2-x for x in 1-5, 10, and 15  
    -  2 PCR replicates per 96 well plate  
    -  2 Replicate 96 well plates
<!--
    -  2 Library Prep Laboratories (barcode and normalization)
        +  barcode and normalization confounded - use picogreen concentration measurements to control for normalization
    -  2 Sequencing laboratories
* Total of 720 samples, 768 including no template PCR controls
-->

* Sequence processing  
    - two commonly used pipelines mothur and qiime SOP  
    - R native pipeline  

* Characterization of sequencing variability  
* Comparison of differential abundance methods  

#### Preliminary Results
* Sample selection  
* 16S PCR qc  
* Normalization/ library prep qc  
* Lib size read counts  


## Aim 2

### Title
Develop a framework for evaluating diversity metrics for 16S rRNA sequence data.  

### Summary
Alpha diversity metrics are commonly used for characterizing and comparing microbial communities. 
These metrics were originally developed for use in macro-ecology, they have been applied to microbial communities often with little regard for the appropriateness of the methods. 
While a number of studies evaluating the alpha and beta diversity methods have been published more often than not methods commonly used by the community remain popular despite evidence to suggestion alternative methods are more appropriate. 
We propose to establish a framework for evaluating diversity metrics  using ecology informed simulated datasets, mock communities, and real datasets including the dataset developed as part of Aim 1.  

### Background and Significance


#### Background
* Objective: evaluate the performance of alpha and beta diversity metrics within the context of 16S marker-gene metagenomics
    - Characterize metric - bias and variance  
* Impact of differences in sample size and variations in methods on values, e.g. how do the metric values compare when the same sample is processed using different methods (both sample processing and data analysis)  
* Intended study results  
    - provide recommendations for diversity metric selection along with information regarding metric limitations and assumptions relative to marker gene sequencing  


#### Significance
* Two applications  
    - assessing diversity metrics suitability  
    - optimizing a bioinformatic pipeline  

### Approach

#### Sub Aims
* We will develop a framework for evaluating diversity metric robustness for well known sources of technical variability.  
* We will develop an analytical approach to predicting the impact of sequencing errors and sequence processing parameters on different $\alpha$ and $\beta$ diversity metrics.  
* We will evaluate the impact of different sequencing and sequence processing methods on alpha and beta diversity metrics using sensitivity analysis.  


#### Approach Details
* Approach use data from environmental, mock communities, and simulated datasets to develop a framework for evaluating diversity metrics  
* Metric Suitability  
    * Evaluate impact of sampling datasets will be subsampled in silico  
value in use of resampling methods to evaluate diversity metrics, e.g. bootstrap and jackknife  
* Evaluate performance based on metric precision  
* Pipeline optimization  
    * process benchmarking datasets using a “on factor at a time” bioinformatic experimental design  
* Evaluate of sequence processing methods - sensitivity analysis  
* Benchmarking datasets  
    * obtain publicly available data that is well annotated with both technical and biological replicates.  This will include both mock communities as well as environmental samples.  


#### Preliminary Results
Need to do  

