<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Measurement Science of Metagenomics</title>
  <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
  <meta name="description" content="Measurement Science of Metagenomics">
  <meta name="generator" content="bookdown 0.0.63 and GitBook 2.6.7">

  <meta property="og:title" content="Measurement Science of Metagenomics" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Measurement Science of Metagenomics" />
  
  
  



<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  TeX: { equationNumbers: { autoNumber: "AMS" } }
});
</script>

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="assessment-framework.html">


<script src="libs/jquery/jquery.min.js"></script>
<link href="libs/gitbook/css/style.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>


  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Metrology of Marker-Gene Metagenomics </a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Metrology of Metagenomics</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#microbiome"><i class="fa fa-check"></i><b>1.1</b> Microbiome</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#measuring-microbiomes"><i class="fa fa-check"></i><b>1.2</b> Measuring Microbiomes</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#metrology"><i class="fa fa-check"></i><b>1.3</b> Metrology</a></li>
<li class="chapter" data-level="1.4" data-path="index.html"><a href="index.html#key-references"><i class="fa fa-check"></i><b>1.4</b> Key references</a><ul>
<li class="chapter" data-level="1.4.1" data-path="index.html"><a href="index.html#measurement-process"><i class="fa fa-check"></i><b>1.4.1</b> Measurement Process</a></li>
<li class="chapter" data-level="1.4.2" data-path="index.html"><a href="index.html#sample-processing-and-sequencing"><i class="fa fa-check"></i><b>1.4.2</b> Sample Processing and Sequencing</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="index.html"><a href="index.html#outline"><i class="fa fa-check"></i><b>1.5</b> Outline</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html"><i class="fa fa-check"></i><b>2</b> 16S rRNA Gene</a><ul>
<li class="chapter" data-level="2.1" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html#general-background-about-16s-rrna-gene"><i class="fa fa-check"></i><b>2.1</b> General background about 16S rRNA Gene</a></li>
<li class="chapter" data-level="2.2" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html#s-rrna-phylogenetics"><i class="fa fa-check"></i><b>2.2</b> 16S rRNA phylogenetics</a></li>
<li class="chapter" data-level="2.3" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html#s-rrna-databases"><i class="fa fa-check"></i><b>2.3</b> 16S rRNA Databases</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="process-characterization.html"><a href="process-characterization.html"><i class="fa fa-check"></i><b>3</b> Process Characterization</a></li>
<li class="chapter" data-level="4" data-path="measurement-process-1.html"><a href="measurement-process-1.html"><i class="fa fa-check"></i><b>4</b> Measurement Process</a><ul>
<li class="chapter" data-level="4.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-to-seq"><i class="fa fa-check"></i><b>4.1</b> Sample to Seq</a><ul>
<li class="chapter" data-level="4.1.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-collection"><i class="fa fa-check"></i><b>4.1.1</b> Sample Collection</a></li>
<li class="chapter" data-level="4.1.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-processing"><i class="fa fa-check"></i><b>4.1.2</b> Sample Processing</a></li>
<li class="chapter" data-level="4.1.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#s-rrna-pcr"><i class="fa fa-check"></i><b>4.1.3</b> 16S rRNA PCR</a></li>
<li class="chapter" data-level="4.1.4" data-path="measurement-process-1.html"><a href="measurement-process-1.html#library-preparation"><i class="fa fa-check"></i><b>4.1.4</b> Library Preparation</a></li>
<li class="chapter" data-level="4.1.5" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sequencing"><i class="fa fa-check"></i><b>4.1.5</b> Sequencing</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#seq-to-otu"><i class="fa fa-check"></i><b>4.2</b> Seq to OTU</a><ul>
<li class="chapter" data-level="4.2.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#preprocessing"><i class="fa fa-check"></i><b>4.2.1</b> Preprocessing</a></li>
<li class="chapter" data-level="4.2.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#otu-assignment"><i class="fa fa-check"></i><b>4.2.2</b> OTU Assignment</a></li>
<li class="chapter" data-level="4.2.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#taxonomic-assignment"><i class="fa fa-check"></i><b>4.2.3</b> Taxonomic Assignment</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#downstream-analysis"><i class="fa fa-check"></i><b>4.3</b> Downstream analysis</a><ul>
<li class="chapter" data-level="4.3.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#differential-abundance"><i class="fa fa-check"></i><b>4.3.1</b> Differential Abundance</a></li>
<li class="chapter" data-level="4.3.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#alpha-diversity"><i class="fa fa-check"></i><b>4.3.2</b> Alpha Diversity</a></li>
<li class="chapter" data-level="4.3.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#beta-diversity"><i class="fa fa-check"></i><b>4.3.3</b> Beta Diversity</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="measurement-process-1.html"><a href="measurement-process-1.html#step-by-step-protocols"><i class="fa fa-check"></i><b>4.4</b> Step-by-Step protocols</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html"><i class="fa fa-check"></i><b>5</b> 16S rRNA Analysis Method Assessment</a><ul>
<li class="chapter" data-level="5.1" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#sample-to-fastq"><i class="fa fa-check"></i><b>5.1</b> Sample to fastq</a></li>
<li class="chapter" data-level="5.2" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#seq-to-otu-1"><i class="fa fa-check"></i><b>5.2</b> Seq to OTU</a><ul>
<li class="chapter" data-level="5.2.1" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#pre-processing"><i class="fa fa-check"></i><b>5.2.1</b> Pre-processing</a></li>
<li class="chapter" data-level="5.2.2" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#chimera-detection"><i class="fa fa-check"></i><b>5.2.2</b> Chimera Detection</a></li>
<li class="chapter" data-level="5.2.3" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#otu-assignment-1"><i class="fa fa-check"></i><b>5.2.3</b> OTU Assignment</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#downstream-analysis-1"><i class="fa fa-check"></i><b>5.3</b> Downstream analysis</a><ul>
<li class="chapter" data-level="5.3.1" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#taxonomic-classification"><i class="fa fa-check"></i><b>5.3.1</b> taxonomic classification</a></li>
<li class="chapter" data-level="5.3.2" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#differential-abundance-1"><i class="fa fa-check"></i><b>5.3.2</b> differential abundance</a></li>
<li class="chapter" data-level="5.3.3" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#ecological-diversity"><i class="fa fa-check"></i><b>5.3.3</b> ecological diversity</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="mixture-study.html"><a href="mixture-study.html"><i class="fa fa-check"></i><b>6</b> Mixture Study</a><ul>
<li class="chapter" data-level="6.1" data-path="mixture-study.html"><a href="mixture-study.html#part-i---development-and-characterization-of-dataset"><i class="fa fa-check"></i><b>6.1</b> Part I - Development and Characterization of Dataset</a></li>
<li class="chapter" data-level="6.2" data-path="mixture-study.html"><a href="mixture-study.html#part-ii-sensitivity-analysis"><i class="fa fa-check"></i><b>6.2</b> Part II Sensitivity Analysis</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="ecological-diversity-robustness.html"><a href="ecological-diversity-robustness.html"><i class="fa fa-check"></i><b>7</b> Ecological Diversity Robustness</a><ul>
<li class="chapter" data-level="7.1" data-path="ecological-diversity-robustness.html"><a href="ecological-diversity-robustness.html#overview"><i class="fa fa-check"></i><b>7.1</b> Overview</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="gold-standard-pipeline.html"><a href="gold-standard-pipeline.html"><i class="fa fa-check"></i><b>8</b> Gold Standard Pipeline</a></li>
<li class="chapter" data-level="9" data-path="assessment-framework.html"><a href="assessment-framework.html"><i class="fa fa-check"></i><b>9</b> Assessment Framework</a></li>
<li class="chapter" data-level="10" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>10</b> References</a></li>
<li class="divider"></li>
<li><a href="https://www.somewebsite.com" target="blank">Published with N8O</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Measurement Science of Metagenomics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1">
<h1><span class="header-section-number">10</span> References</h1>

<div id="refs" class="references">
<div id="ref-Albertsen2015">
<p>Albertsen, Mads, Søren M Karst, Anja S Ziegler, Rasmus H Kirkegaard, and Per H Nielsen. 2015. “Back to Basics - The Influence of DNA Extraction and Primer Choice on Phylogenetic Analysis of Activated Sludge Communities.” <em>PloS One</em> 10 (7). Public Library of Science: e0132783. doi:<a href="https://doi.org/10.1371/journal.pone.0132783">10.1371/journal.pone.0132783</a>.</p>
</div>
<div id="ref-Blazewicz2013">
<p>Blazewicz, Steven J, Romain L Barnard, Rebecca A Daly, and Mary K Firestone. 2013. “Evaluating rRNA as an indicator of microbial activity in environmental communities: limitations and uses.” <em>The ISME Journal</em> 7 (11). International Society for Microbial Ecology: 2061–8. doi:<a href="https://doi.org/10.1038/ismej.2013.102">10.1038/ismej.2013.102</a>.</p>
</div>
<div id="ref-Brooks2015">
<p>Brooks, J Paul, David J Edwards, Michael D Harwich, Maria C Rivera, Jennifer M Fettweis, Myrna G Serrano, Robert A Reris, et al. 2015. “The truth about metagenomics: quantifying and counteracting bias in 16S rRNA studies.” <em>BMC Microbiology</em> 15 (1): 66. doi:<a href="https://doi.org/10.1186/s12866-015-0351-6">10.1186/s12866-015-0351-6</a>.</p>
</div>
<div id="ref-Bunge2014">
<p>Bunge, John, Amy Willis, and Fiona Walsh. 2014. “Estimating the Number of Species in Microbial Diversity Studies.” <em>Annual Review of Statistics and Its Application</em> 1 (1): 427–45. doi:<a href="https://doi.org/10.1146/annurev-statistics-022513-115654">10.1146/annurev-statistics-022513-115654</a>.</p>
</div>
<div id="ref-Cangelosi2014">
<p>Cangelosi, Gerard A, and John S Meschke. 2014. “Dead or alive: molecular assessment of microbial viability.” <em>Applied and Environmental Microbiology</em> 80 (19): 5884–91. doi:<a href="https://doi.org/10.1128/AEM.01763-14">10.1128/AEM.01763-14</a>.</p>
</div>
<div id="ref-Carini2016">
<p>Carini, Paul, Patrick J Marsden, Jonathan W Leff, Emily E Morgan, Michael S Strickland, and Noah Fierer. 2016. “Relic DNA is abundant in soil and obscures estimates of soil microbial diversity.” Cold Spring Harbor Labs Journals. doi:<a href="https://doi.org/10.1101/043372">10.1101/043372</a>.</p>
</div>
<div id="ref-Carrigg2007">
<p>Carrigg, Cora, Olivia Rice, Siobhán Kavanagh, Gavin Collins, and Vincent O’Flaherty. 2007. “DNA extraction method affects microbial community profiles from soils and sediment.” <em>Applied Microbiology and Biotechnology</em> 77 (4): 955–64. doi:<a href="https://doi.org/10.1007/s00253-007-1219-y">10.1007/s00253-007-1219-y</a>.</p>
</div>
<div id="ref-Chao2014">
<p>Chao, Anne, Nicholas J. Gotelli, T. C. Hsieh, Elizabeth L. Sander, K. H. Ma, Robert K. Colwell, and Aaron M. Ellison. 2014. “Rarefaction and extrapolation with Hill numbers: A framework for sampling and estimation in species diversity studies.” <em>Ecological Monographs</em> 84 (1): 45–67. doi:<a href="https://doi.org/10.1890/13-0133.1">10.1890/13-0133.1</a>.</p>
</div>
<div id="ref-Chen2013">
<p>Chen, Wei, Clarence K. Zhang, Yongmei Cheng, Shaowu Zhang, and Hongyu Zhao. 2013. “A Comparison of Methods for Clustering 16S rRNA Sequences into OTUs.” <em>PLoS ONE</em> 8 (8). doi:<a href="https://doi.org/10.1371/journal.pone.0070837">10.1371/journal.pone.0070837</a>.</p>
</div>
<div id="ref-Chiu2015">
<p>Chiu, Chun-Huo, and Anne Chao. 2015. “Estimating and comparing microbial diversity in the presence of sequencing errors.” <em>PeerJ</em>, 1–13. doi:<a href="https://doi.org/10.7287/peerj.preprints.11">10.7287/peerj.preprints.11</a>.</p>
</div>
<div id="ref-Clooney2016">
<p>Clooney, Adam G, Fiona Fouhy, Roy D Sleator, Aisling O’ Driscoll, Catherine Stanton, Paul D Cotter, and Marcus J Claesson. 2016. “Comparing Apples and Oranges?: Next Generation Sequencing and Its Impact on Microbiome Analysis.” <em>PloS One</em> 11 (2): e0148028. doi:<a href="https://doi.org/10.1371/journal.pone.0148028">10.1371/journal.pone.0148028</a>.</p>
</div>
<div id="ref-Claesson2010">
<p>“Comparison of two next-generation sequencing technologies for resolving highly complex microbiota composition using tandem variable 16S rRNA gene regions.” 2010. <em>Nucleic Acids Research</em> 38 (22). Department of Microbiology, University College Cork, Cork, Ireland. mclaesson@bioinfo.ucc.ie; Oxford University Press: e200. doi:<a href="https://doi.org/10.1093/nar/gkq873">10.1093/nar/gkq873</a>.</p>
</div>
<div id="ref-Amore2016">
<p>D’Amore, Rosalinda, Umer Zeeshan Ijaz, Melanie Schirmer, John G Kenny, Richard Gregory, Alistair C Darby, Christopher Quince, et al. 2016. “A comprehensive benchmarking study of protocols and sequencing platforms for 16S rRNA community profiling.” <em>BMC Genomics</em> 17. BMC Genomics: 1–40. doi:<a href="https://doi.org/10.1186/s12864-015-2194-9">10.1186/s12864-015-2194-9</a>.</p>
</div>
<div id="ref-Engelbrektson2010">
<p>Engelbrektson, Anna, Victor Kunin, Kelly C Wrighton, Natasha Zvenigorodsky, Feng Chen, Howard Ochman, and Philip Hugenholtz. 2010. “Experimental factors affecting PCR-based estimates of microbial species richness and evenness.” <em>The ISME Journal</em> 4 (5). Nature Publishing Group: 642–7. doi:<a href="https://doi.org/10.1038/ismej.2009.153">10.1038/ismej.2009.153</a>.</p>
</div>
<div id="ref-Esling2015">
<p>Esling, Philippe, Franck Lejzerowicz, and Jan Pawlowski. 2015. “Accurate multiplexing and filtering for high-throughput amplicon-sequencing.” <em>Nucleic Acids Research</em> 43 (10): 2513–24. doi:<a href="https://doi.org/10.1093/nar/gkv107">10.1093/nar/gkv107</a>.</p>
</div>
<div id="ref-Ettenauer2012">
<p>Ettenauer, Jörg D, Guadalupe Piñar, Ksenija Lopandic, Bernhard Spangl, Günther Ellersdorfer, Christian Voitl, and Katja Sterflinger. 2012. “Microbes on building materials - Evaluation of DNA extraction protocols as common basis for molecular analysis.” <em>The Science of the Total Environment</em> 439C (October). Elsevier B.V.: 44–53. doi:<a href="https://doi.org/10.1016/j.scitotenv.2012.09.005">10.1016/j.scitotenv.2012.09.005</a>.</p>
</div>
<div id="ref-Flores2015">
<p>Flores, Roberto, Jianxin Shi, Guoqin Yu, Bing Ma, Jacques Ravel, James J. Goedert, and Rashmi Sinha. 2015. “Collection media and delayed freezing effects on microbial composition of human stool.” <em>Microbiome</em> 3 (1). BioMed Central Ltd: 33. doi:<a href="https://doi.org/10.1186/s40168-015-0092-7">10.1186/s40168-015-0092-7</a>.</p>
</div>
<div id="ref-Ghodsi2011">
<p>Ghodsi, Mohammadreza, Bo Liu, and Mihai Pop. 2011. “DNACLUST: accurate and efficient clustering of phylogenetic marker genes.” <em>BMC Bioinformatics</em> 12 (1). BioMed Central Ltd: 271. doi:<a href="https://doi.org/10.1186/1471-2105-12-271">10.1186/1471-2105-12-271</a>.</p>
</div>
<div id="ref-Goodrich2014">
<p>Goodrich, Julia K., Sara C. Di Rienzi, Angela C. Poole, Omry Koren, William A. Walters, J. Gregory Caporaso, Rob Knight, and Ruth E. Ley. 2014. “Conducting a Microbiome Study.” <em>Cell</em> 158 (2). Elsevier Inc.: 250–62. doi:<a href="https://doi.org/10.1016/j.cell.2014.06.037">10.1016/j.cell.2014.06.037</a>.</p>
</div>
<div id="ref-He2015">
<p>He, Yan, J Gregory Caporaso, Xiao-Tao Jiang, Hua-Fang Sheng, Susan M Huse, Jai Ram Rideout, Robert C Edgar, et al. 2015. “Stability of operational taxonomic units: an important but neglected property for analyzing microbial diversity.” <em>Microbiome</em> 3 (1). ??? 20. doi:<a href="https://doi.org/10.1186/s40168-015-0081-x">10.1186/s40168-015-0081-x</a>.</p>
</div>
<div id="ref-Huse2010">
<p>Huse, Susan M, David Mark Welch, Hilary G Morrison, and Mitchell L Sogin. 2010. “Ironing out the wrinkles in the rare biosphere through improved OTU clustering.” <em>Environmental Microbiology</em> 12 (7): 1889–98. doi:<a href="https://doi.org/10.1111/j.1462-2920.2010.02193.x">10.1111/j.1462-2920.2010.02193.x</a>.</p>
</div>
<div id="ref-Jonsson2016">
<p>Jonsson, Viktor, Tobias Österlund, Olle Nerman, and Erik Kristiansson. 2016. “Statistical evaluation of methods for identification of differentially abundant genes in comparative metagenomics.” <em>BMC Genomics</em> 17 (1). BioMed Central: 78. doi:<a href="https://doi.org/10.1186/s12864-016-2386-y">10.1186/s12864-016-2386-y</a>.</p>
</div>
<div id="ref-Kennedy2014">
<p>Kennedy, Katherine, Michael W. Hall, Michael D J Lynch, Gabriel Moreno-Hagelsieb, and Josh D. Neufeld. 2014. “Evaluating bias of Illumina-based bacterial 16S rRNA gene profiles.” <em>Applied and Environmental Microbiology</em> 80 (18): 5717–22. doi:<a href="https://doi.org/10.1128/AEM.01451-14">10.1128/AEM.01451-14</a>.</p>
</div>
<div id="ref-Kopylova2014">
<p>Kopylova, Evguenia, Jose A Navas-molina, Céline Mercier, and Zech Xu. 2014. “Open-Source Sequence Clustering Methods Improve the State Of the Art.” <em>MSystems</em> 1 (1): 1–16. doi:<a href="https://doi.org/10.1128/mSystems.00003-15.Editor">10.1128/mSystems.00003-15.Editor</a>.</p>
</div>
<div id="ref-Kozich2013">
<p>Kozich, James J., Sarah L. Westcott, Nielson T. Baxter, Sarah K. Highlander, and Patrick D. Schloss. 2013. “Development of a Dual-Index Sequencing Strategy and Curation Pipeline for Analyzing Amplicon Sequence Data on the MiSeq Illumina Sequencing Platform.” <em>Applied and Environmental Microbiology</em> 79 (17): 5112–20. doi:<a href="https://doi.org/10.1128/AEM.01043-13">10.1128/AEM.01043-13</a>.</p>
</div>
<div id="ref-Kunin2010">
<p>Kunin, Victor, Anna Engelbrektson, Howard Ochman, and Philip Hugenholtz. 2010. “Wrinkles in the rare biosphere: pyrosequencing errors can lead to artificial inflation of diversity estimates.” <em>Environmental Microbiology</em> 12 (1): 118–23. doi:<a href="https://doi.org/10.1111/j.1462-2920.2009.02051.x">10.1111/j.1462-2920.2009.02051.x</a>.</p>
</div>
<div id="ref-masella2012pandaseq">
<p>Masella, Andre P, Andrea K Bartram, Jakub M Truszkowski, Daniel G Brown, and Josh D Neufeld. 2012. “PANDAseq: Paired-End Assembler for Illumina Sequences.” <em>BMC Bioinformatics</em> 13 (1). BioMed Central Ltd: 31.</p>
</div>
<div id="ref-Olson2012">
<p>Olson, Nathan D, and Jayne B Morrow. 2012. “DNA extract characterization process for microbial detection methods development and validation.” <em>BMC Research Notes</em> 5 (1). BMC Research Notes: 668. doi:<a href="https://doi.org/10.1186/1756-0500-5-668">10.1186/1756-0500-5-668</a>.</p>
</div>
<div id="ref-Ricotta2007">
<p>Ricotta, Carlo. 2007. “A semantic taxonomy for diversity measures.” <em>Acta Biotheoretica</em> 55 (1): 23–33. doi:<a href="https://doi.org/10.1007/s10441-007-9008-7">10.1007/s10441-007-9008-7</a>.</p>
</div>
<div id="ref-Schirmer2015">
<p>Schirmer, Melanie, Umer Z. Ijaz, Rosalinda D’Amore, Neil Hall, William T. Sloan, and Christopher Quince. 2015. “Insight into biases and sequencing errors for amplicon sequencing with the Illumina MiSeq platform.” <em>Nucleic Acids Research</em> 43 (6). doi:<a href="https://doi.org/10.1093/nar/gku1341">10.1093/nar/gku1341</a>.</p>
</div>
<div id="ref-Schloss2016">
<p>Schloss, Patrick D. 2016. “Application of database-independent approach to assess the quality of OTU picking methods.”</p>
</div>
<div id="ref-Schloss2011">
<p>Schloss, Patrick D., and Sarah L. Westcott. 2011. “Assessing and Improving Methods Used in Operational Taxonomic Unit-Based Approaches for 16S rRNA Gene Sequence Analysis.” <em>Applied and Environmental Microbiology</em> 77 (10): 3219–26. doi:<a href="https://doi.org/10.1128/AEM.02810-10">10.1128/AEM.02810-10</a>.</p>
</div>
<div id="ref-Schmidt2015">
<p>Schmidt, Thomas S B, Jo??o F. Matias Rodrigues, and Christian von Mering. 2015. “Limits to robustness and reproducibility in the demarcation of operational taxonomic units.” <em>Environmental Microbiology</em> 17 (5): 1689–1706. doi:<a href="https://doi.org/10.1111/1462-2920.12610">10.1111/1462-2920.12610</a>.</p>
</div>
<div id="ref-Tuomisto2012">
<p>Tuomisto, Hanna. 2012. “An updated consumer’s guide to evenness and related indices.” <em>Oikos</em> 121 (8): 1203–18. doi:<a href="https://doi.org/10.1111/j.1600-0706.2011.19897.x">10.1111/j.1600-0706.2011.19897.x</a>.</p>
</div>
<div id="ref-Westcott2015">
<p>Westcott, Sarah L, and Patrick D Schloss. 2015. “De novo clustering methods out-perform reference-based methods for assigning 16S rRNA gene sequences to operational taxonomic units.” <em>PeerJ</em> 3: e1487. doi:<a href="https://doi.org/10.7287/peerj.preprints.1466v1">10.7287/peerj.preprints.1466v1</a>.</p>
</div>
<div id="ref-Wu2013">
<p>Wu, Dongying, Ladan Doroud, and Jonathan a. Eisen. 2013. “TreeOTU Operational Taxonomic Unit Classification Based on Phylogenetic Trees.” <em>Arxiv.Org</em>, 23. <a href="http://arxiv.org/abs/1308.6333" class="uri">http://arxiv.org/abs/1308.6333</a>.</p>
</div>
<div id="ref-Yu2015">
<p>Yu, Guoqin, Doug Fadrosh, James J. Goedert, Jacques Ravel, and Alisa M. Goldstein. 2015. “Nested PCR Biases in Interpreting Microbial Community Structure in 16S rRNA Gene Sequence Datasets.” <em>Plos One</em> 10 (7): e0132253. doi:<a href="https://doi.org/10.1371/journal.pone.0132253">10.1371/journal.pone.0132253</a>.</p>
</div>
<div id="ref-zhang2014pear">
<p>Zhang, Jiajie, Kassian Kobert, Tomáš Flouri, and Alexandros Stamatakis. 2014. “PEAR: A Fast and Accurate Illumina Paired-End ReAd MergeR.” <em>Bioinformatics</em> 30 (5). Oxford Univ Press: 614–20.</p>
</div>
<div id="ref-Zhou2011">
<p>Zhou, Jizhong, Liyou Wu, Ye Deng, Xiaoyang Zhi, Yi-Huei Jiang, Qichao Tu, Jianping Xie, Joy D Van Nostrand, Zhili He, and Yunfeng Yang. 2011. “Reproducibility and quantitation of amplicon sequencing-based detection.” <em>The ISME Journal</em> 5 (8). Nature Publishing Group: 1303–13. doi:<a href="https://doi.org/10.1038/ismej.2011.11">10.1038/ismej.2011.11</a>.</p>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="assessment-framework.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>


<script src="libs/gitbook/js/app.js"></script>
<script src="libs/gitbook/js/lunr.js"></script>
<script src="libs/gitbook/js/plugin-search.js"></script>
<script src="libs/gitbook/js/plugin-sharing.js"></script>
<script src="libs/gitbook/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook/js/plugin-bookdown.js"></script>
<script src="libs/gitbook/js/jquery.highlight.js"></script>
<script>
require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/inst/examples/80-References.Rmd",
"text": "Edit"
},
"download": null,
"toc": {
"collapse": false
},
"toolbar": {
"position": "fixed"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
