<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Measurement Science of Metagenomics</title>
  <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
  <meta name="description" content="Measurement Science of Metagenomics">
  <meta name="generator" content="bookdown 0.0.72 and GitBook 2.6.7">

  <meta property="og:title" content="Measurement Science of Metagenomics" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Measurement Science of Metagenomics" />
  
  
  



<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  TeX: { equationNumbers: { autoNumber: "AMS" } }
});
</script>

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="measurement-process-1.html">
<link rel="next" href="mixture-study.html">

<script src="libs/jquery/jquery.min.js"></script>
<link href="libs/gitbook/css/style.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>


  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Metrology of Marker-Gene Metagenomics </a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Metrology of Metagenomics</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#microbiome"><i class="fa fa-check"></i><b>1.1</b> Microbiome</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#measuring-microbiomes"><i class="fa fa-check"></i><b>1.2</b> Measuring Microbiomes</a></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#metrology"><i class="fa fa-check"></i><b>1.3</b> Metrology</a></li>
<li class="chapter" data-level="1.4" data-path="index.html"><a href="index.html#key-references"><i class="fa fa-check"></i><b>1.4</b> Key references</a><ul>
<li class="chapter" data-level="1.4.1" data-path="index.html"><a href="index.html#measurement-process"><i class="fa fa-check"></i><b>1.4.1</b> Measurement Process</a></li>
<li class="chapter" data-level="1.4.2" data-path="index.html"><a href="index.html#sample-processing-and-sequencing"><i class="fa fa-check"></i><b>1.4.2</b> Sample Processing and Sequencing</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="index.html"><a href="index.html#outline"><i class="fa fa-check"></i><b>1.5</b> Outline</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html"><i class="fa fa-check"></i><b>2</b> 16S rRNA Gene</a><ul>
<li class="chapter" data-level="2.1" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html#general-background-about-16s-rrna-gene"><i class="fa fa-check"></i><b>2.1</b> General background about 16S rRNA Gene</a></li>
<li class="chapter" data-level="2.2" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html#s-rrna-phylogenetics"><i class="fa fa-check"></i><b>2.2</b> 16S rRNA phylogenetics</a></li>
<li class="chapter" data-level="2.3" data-path="s-rrna-gene.html"><a href="s-rrna-gene.html#s-rrna-databases"><i class="fa fa-check"></i><b>2.3</b> 16S rRNA Databases</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="process-characterization.html"><a href="process-characterization.html"><i class="fa fa-check"></i><b>3</b> Process Characterization</a></li>
<li class="chapter" data-level="4" data-path="measurement-process-1.html"><a href="measurement-process-1.html"><i class="fa fa-check"></i><b>4</b> Measurement Process</a><ul>
<li class="chapter" data-level="4.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-to-seq"><i class="fa fa-check"></i><b>4.1</b> Sample to Seq</a><ul>
<li class="chapter" data-level="4.1.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-collection"><i class="fa fa-check"></i><b>4.1.1</b> Sample Collection</a></li>
<li class="chapter" data-level="4.1.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sample-processing"><i class="fa fa-check"></i><b>4.1.2</b> Sample Processing</a></li>
<li class="chapter" data-level="4.1.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#s-rrna-pcr"><i class="fa fa-check"></i><b>4.1.3</b> 16S rRNA PCR</a></li>
<li class="chapter" data-level="4.1.4" data-path="measurement-process-1.html"><a href="measurement-process-1.html#library-preparation"><i class="fa fa-check"></i><b>4.1.4</b> Library Preparation</a></li>
<li class="chapter" data-level="4.1.5" data-path="measurement-process-1.html"><a href="measurement-process-1.html#sequencing"><i class="fa fa-check"></i><b>4.1.5</b> Sequencing</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#seq-to-otu"><i class="fa fa-check"></i><b>4.2</b> Seq to OTU</a><ul>
<li class="chapter" data-level="4.2.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#preprocessing"><i class="fa fa-check"></i><b>4.2.1</b> Preprocessing</a></li>
<li class="chapter" data-level="4.2.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#otu-assignment"><i class="fa fa-check"></i><b>4.2.2</b> OTU Assignment</a></li>
<li class="chapter" data-level="4.2.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#taxonomic-assignment"><i class="fa fa-check"></i><b>4.2.3</b> Taxonomic Assignment</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#downstream-analysis"><i class="fa fa-check"></i><b>4.3</b> Downstream analysis</a><ul>
<li class="chapter" data-level="4.3.1" data-path="measurement-process-1.html"><a href="measurement-process-1.html#differential-abundance"><i class="fa fa-check"></i><b>4.3.1</b> Differential Abundance</a></li>
<li class="chapter" data-level="4.3.2" data-path="measurement-process-1.html"><a href="measurement-process-1.html#alpha-diversity"><i class="fa fa-check"></i><b>4.3.2</b> Alpha Diversity</a></li>
<li class="chapter" data-level="4.3.3" data-path="measurement-process-1.html"><a href="measurement-process-1.html#beta-diversity"><i class="fa fa-check"></i><b>4.3.3</b> Beta Diversity</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="measurement-process-1.html"><a href="measurement-process-1.html#step-by-step-protocols"><i class="fa fa-check"></i><b>4.4</b> Step-by-Step protocols</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html"><i class="fa fa-check"></i><b>5</b> 16S rRNA Analysis Method Assessment</a><ul>
<li class="chapter" data-level="5.1" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#sample-to-fastq"><i class="fa fa-check"></i><b>5.1</b> Sample to fastq</a></li>
<li class="chapter" data-level="5.2" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#seq-to-otu-1"><i class="fa fa-check"></i><b>5.2</b> Seq to OTU</a><ul>
<li class="chapter" data-level="5.2.1" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#pre-processing"><i class="fa fa-check"></i><b>5.2.1</b> Pre-processing</a></li>
<li class="chapter" data-level="5.2.2" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#chimera-detection"><i class="fa fa-check"></i><b>5.2.2</b> Chimera Detection</a></li>
<li class="chapter" data-level="5.2.3" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#otu-assignment-1"><i class="fa fa-check"></i><b>5.2.3</b> OTU Assignment</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#downstream-analysis-1"><i class="fa fa-check"></i><b>5.3</b> Downstream analysis</a><ul>
<li class="chapter" data-level="5.3.1" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#taxonomic-classification"><i class="fa fa-check"></i><b>5.3.1</b> taxonomic classification</a></li>
<li class="chapter" data-level="5.3.2" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#differential-abundance-1"><i class="fa fa-check"></i><b>5.3.2</b> differential abundance</a></li>
<li class="chapter" data-level="5.3.3" data-path="s-rrna-analysis-method-assessment.html"><a href="s-rrna-analysis-method-assessment.html#ecological-diversity"><i class="fa fa-check"></i><b>5.3.3</b> ecological diversity</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="mixture-study.html"><a href="mixture-study.html"><i class="fa fa-check"></i><b>6</b> Mixture Study</a><ul>
<li class="chapter" data-level="6.1" data-path="mixture-study.html"><a href="mixture-study.html#part-i---development-and-characterization-of-dataset"><i class="fa fa-check"></i><b>6.1</b> Part I - Development and Characterization of Dataset</a></li>
<li class="chapter" data-level="6.2" data-path="mixture-study.html"><a href="mixture-study.html#part-ii-sensitivity-analysis"><i class="fa fa-check"></i><b>6.2</b> Part II Sensitivity Analysis</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="ecological-diversity-robustness.html"><a href="ecological-diversity-robustness.html"><i class="fa fa-check"></i><b>7</b> Ecological Diversity Robustness</a><ul>
<li class="chapter" data-level="7.1" data-path="ecological-diversity-robustness.html"><a href="ecological-diversity-robustness.html#overview"><i class="fa fa-check"></i><b>7.1</b> Overview</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="gold-standard-pipeline.html"><a href="gold-standard-pipeline.html"><i class="fa fa-check"></i><b>8</b> Gold Standard Pipeline</a></li>
<li class="chapter" data-level="9" data-path="assessment-framework.html"><a href="assessment-framework.html"><i class="fa fa-check"></i><b>9</b> Assessment Framework</a></li>
<li class="chapter" data-level="10" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>10</b> References</a></li>
<li class="divider"></li>
<li><a href="https://www.somewebsite.com" target="blank">Published with N8O</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Measurement Science of Metagenomics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="s-rrna-analysis-method-assessment" class="section level1">
<h1><span class="header-section-number">5</span> 16S rRNA Analysis Method Assessment</h1>
<p>Method steps<br />
1. Sample to Seq 1. Seq to OTU<br />
2. Downstream analysis 3. Taxonomic Classification<br />
 4. Differential Abundance<br />
 5. Ecological Diversity</p>
<div id="sample-to-fastq" class="section level2">
<h2><span class="header-section-number">5.1</span> Sample to fastq</h2>
<ul>
<li>Benchmarking
<ul>
<li>using <em>in vitro</em> mock communities (e.g. cells, DNA, or PCR products)<br />
</li>
<li>technical replicates?<br />
</li>
<li>well characterized benchmarking samples - similar to Genomic RMs (deep sequenced multiple methods)<br />
</li>
<li>Mixture/ titrations<br />
</li>
<li>Standard additions<br />
</li>
<li>Dilution to extinction?<br />
</li>
</ul></li>
<li>Experimental design<br />
</li>
<li><span class="citation">Brooks et al. (<a href="#ref-Brooks2015">2015</a>)</span> generated 80 mock communities using different combinations of 7 bacterial strains generating mixtures of cells, DNA, and PCR products to characterize the contributions of different steps in the measurment process. Evalualted the results using the mixture effect model.</li>
</ul>
</div>
<div id="seq-to-otu-1" class="section level2">
<h2><span class="header-section-number">5.2</span> Seq to OTU</h2>
<div id="pre-processing" class="section level3">
<h3><span class="header-section-number">5.2.1</span> Pre-processing</h3>
<ul>
<li>Fastq Quality Assessment
<ul>
<li>fastqc<br />
</li>
<li>R ShortRead<br />
</li>
<li>R Rqc<br />
</li>
<li>Issues
<ul>
<li>method developed for evaluating individual datasets - metagenomics large numbers of samples<br />
</li>
<li>read level information - how to best seperate out reads of different quality
<ul>
<li>DADA2 requires sequences all of the same length, good to be able to analyze on a per reads level
<ul>
<li>Huse 454 work showing that multiple errors present in some reads wheresas other reads had no errors<br />
</li>
<li>Want to be able to identify error free reads for OTU calling then use all reads for abundance estimates<br />
</li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
<li>Single org samples<br />
</li>
<li>Data sets from sequencing individual organisms using a standard 16S metagenomic pipeline had been used to evaluate sequencing errors and optimize filtering parameters.
<ul>
<li><span class="citation">Kunin et al. (<a href="#ref-Kunin2010">2010</a>)</span> evaluated the sequencing error rates and use of different filtering thresholds for 454 pyrosequencing of an <em>E. coli</em> isolated.<br />
Schloss, P. D. (2010). The Effects of Alignment Quality, Distance Calculation Method, Sequence Filtering, and Region on the Analysis of 16S rRNA Gene-Based Studies. PLoS Comput Biol, 6(7), e1000844. <a href="http://doi.org/10.1371/journal.pcbi.1000844" class="uri">http://doi.org/10.1371/journal.pcbi.1000844</a></li>
</ul></li>
<li>Multiple Org
<ul>
<li><span class="citation">Schirmer et al. (<a href="#ref-Schirmer2015">2015</a>)</span> using a mock community of 59 strains (49 bacteria and 10 archaea) evaluated differnt sequence processing methods and parameters for read filtering and quality trimming, merging paired end reads, and error correction. Methods were evaluated based on error rate reduction.</li>
</ul></li>
</ul>
<p>Albanese, D., Fontana, P., De Filippo, C., Cavalieri, D., &amp; Donati, C. (2015). MICCA: a complete and accurate software for taxonomic profiling of metagenomic data. Scientific Reports, 5, 9743. <a href="http://doi.org/10.1038/srep09743" class="uri">http://doi.org/10.1038/srep09743</a><br />
* Evaluates preprocessing % read passing filtering, chimeras, and redundant OTUs, relative abundance</p>
<p>Gaspar, J. M., &amp; Thomas, W. K. (2013). Assessing the consequences of denoising marker-based metagenomic data. PloS One, 8(3), e60458. <a href="http://doi.org/10.1371/journal.pone.0060458" class="uri">http://doi.org/10.1371/journal.pone.0060458</a><br />
* Evaluation of denoising and chimera detection methods for 454 data</p>
</div>
<div id="chimera-detection" class="section level3">
<h3><span class="header-section-number">5.2.2</span> Chimera Detection</h3>
<p>Mysara, M., Saeys, Y., Leys, N., Raes, J., &amp; Monsieurs, P. (2015). CATCh, an Ensemble Classifier for Chimera Detection in 16S rRNA Sequencing Studies. Applied and Environmental Microbiology, 81(December), 1573–1584. <a href="http://doi.org/10.1128/AEM.02896-14" class="uri">http://doi.org/10.1128/AEM.02896-14</a></p>
<p>Beblur - <a href="https://github.com/ekopylova/deblur" class="uri">https://github.com/ekopylova/deblur</a></p>
</div>
<div id="otu-assignment-1" class="section level3">
<h3><span class="header-section-number">5.2.3</span> OTU Assignment</h3>
<ul>
<li>De novo clustering has been shown to be more accurate <span class="citation">(S. L. Westcott and Schloss <a href="#ref-Westcott2015">2015</a>)</span><br />
</li>
<li>Types of datasets used to assess benchmarking methods: simulated data, mock communities, environmental datasests</li>
<li>Approaches to evaluating clustering methods
<ul>
<li>Cluster stability - similarity in clustering results when varying input data<br />
</li>
<li>Cluster reproducibility - similarity in clustering results between methods<br />
</li>
<li>Number of artifact OTUs - number of observed OTUs relative to expected
<ul>
<li>Evaluating clustering methods using simulated and mock community clustering data provide a level of truth (expectation of the correct result).</li>
</ul></li>
</ul></li>
</ul>
<div id="summary-of-cluster-method-assessment-papers" class="section level4">
<h4><span class="header-section-number">5.2.3.1</span> Summary of Cluster Method Assessment Papers</h4>
<ul>
<li><span class="citation">Kopylova et al. (<a href="#ref-Kopylova2014">2014</a>)</span> used a combination of simulated, mock communities, and data from environmental samples to compare clustering methods.
<ul>
<li>They evaluated the performance of the clustering methods using F-measure and phylogenetic distance.<br />
</li>
<li>For environment datasets the clustering methods were evaluated based on the disimilarity of samples for a dataset based on the sum of the squared deviation for UniFrac PCoA (Procrustes M<sup>2</sup>) and similarity to UCLUST (Pearson’s correlation). <strong>NOTE</strong> Still note sure what they are evaluating for Procrustes.<br />
</li>
<li>The authors excluded singletons from there analysis, while this does help to eliminate spurious OTUs, the method may also exclude rare taxa.<br />
</li>
</ul></li>
<li><span class="citation">Schloss (<a href="#ref-Schloss2016">2016</a>)</span> argues that using simulated and mock communities is not representative of real environmental samples. The authors additionally, argue the approach used by <span class="citation">Kopylova et al. (<a href="#ref-Kopylova2014">2014</a>)</span> to evaluating clustering methods confounds the impact of read filtering with clustering methods.
<ul>
<li>The author’s evaluated environmental datasets using Mathew’s correlation coefficient, for a truth table (TP, TN, FP, FN) based on distance between sequences in the same and different clusters.
<ul>
<li>This method for evaluating clusters was first used in <span class="citation">(Schloss and Westcott <a href="#ref-Schloss2011">2011</a>)</span><br />
</li>
</ul></li>
<li>Average neighbor performed the best based the MCC based assessment. However, this approach to evaluation is biased towards this method as the results are strictly assessed based on the sequence distance criteria. Other clustering methods that attempt to address more nuianced issues with clustering, sequencing error and variability in sequence diversity allow for differences in distances within and between clusters.<br />
</li>
<li>This approach to cluster evaluation is particularly suited for algorithms based on analytical proofs.</li>
<li>For example the DNACluster clustering algorithm employs a heuristic based on a proof that for any given cluster the distance between two sequences in the cluster is less than the defined threshold value and no sequence is closer to another cluster center than the center of the cluster it is assigned to <span class="citation">(Ghodsi, Liu, and Pop <a href="#ref-Ghodsi2011">2011</a>)</span>.
<ul>
<li><span class="citation">Ghodsi, Liu, and Pop (<a href="#ref-Ghodsi2011">2011</a>)</span> used a similar approach to validate their clustering method, though only focusing on the within cluster pairwise distance.</li>
</ul></li>
</ul></li>
<li>Cluster Robustness is another attribute for evaluating clustering methods <span class="citation">(Y. He et al. <a href="#ref-He2015">2015</a>)</span>.
<ul>
<li>Cluster robustness is the reproduciblity of the OTU table generated by a clustering algorith.<br />
</li>
<li>Cluster robustness was evaluated based on:
<ul>
<li>Unstable sequences - sequences represented by different centroids</li>
<li>Unstable OTUs - OTUs where membership was impacted by the number of sequences in a dataset.</li>
<li>using Mathew’s correlation coefficient
<ul>
<li>Truth Table definitions
<ul>
<li>TP if two sequences clustered <strong>together</strong> for the full and subsampled dataset<br />
</li>
<li>TN if two sequences clustered <strong>separately</strong> for the full and subsampled<br />
</li>
<li>FN two sequences clustered <strong>together</strong> in full but not subsampled.<br />
</li>
<li>FP two sequences clustered <strong>separately</strong> in full but together in subsampled dataset.</li>
</ul></li>
<li>The MCC method uses the full dataset at the true value to benchmark the subsampled dataset against.<br />
</li>
<li><strong>NOTE</strong> Is this a valid assumption for clustering?<br />
</li>
</ul></li>
<li>Impact of cluster stablity on Rarefaction curves, PCoA with Bray-Curtis and UniFrac.<br />
</li>
<li>Tested for significance using ADNOIS, a non-parametric test measuring effect size, the amount of the observed variance explained by metadata variables. <strong>NEED REF</strong> - See vegan manual for potenial references.</li>
</ul></li>
<li>The resulting OTU table is dependent the number of sequences clustered.<br />
</li>
<li>For some methods the OTU table is dependent on the order in which the sequences are provided <span class="citation">(Ghodsi, Liu, and Pop <a href="#ref-Ghodsi2011">2011</a>)</span>.<br />
</li>
<li>Closed reference clustering was the only method the produced stable clusters.<br />
</li>
<li>This method discards sequences that do not cluster with reference sequences.<br />
</li>
<li>Open-reference cluster, which performs <em>de novo</em> clustering of unmatched sequences provides a more robust alternative clustering method to <em>de novo</em> clustering while allowing for the diversity of novel OTUs.<br />
</li>
</ul></li>
<li>Cluster method robustness can also be calculated using cross validation<span class="citation">(W. Chen et al. <a href="#ref-Chen2013">2013</a>)</span>.
<ul>
<li><span class="citation">W. Chen et al. (<a href="#ref-Chen2013">2013</a>)</span> subsampled the data to 90% five times and compared the clusters between replicates.<br />
</li>
<li>Methods were evaluated using precision, recall, and NID
<ul>
<li>precision - measure of cluster homogeneity<br />
</li>
<li>recall - measure of cluster completeness<br />
</li>
<li>NID - assess cluster globally???<br />
</li>
</ul></li>
<li>See assessment of cluster quality for metric definitions<br />
</li>
</ul></li>
<li>Cluster robustness can also be evaluated based on threshold and 16S region <span class="citation">(Schmidt, Matias Rodrigues, and Mering <a href="#ref-Schmidt2015">2015</a>)</span>.
<ul>
<li>The authors focus their assessment more on the comparability of results among different methods and note the importances of reproducibility.<br />
</li>
<li>Quote from paper “how robust are biological findings to the choice of clustering method? We found that OTU demarcation may indeed be replicable: different methods provided (almost) identical partitions when twice clustering the exact same sets of sequences, but in randomized order (Fig. 4, diagonals). However, trends in reproducibility were less clear.”<br />
</li>
</ul></li>
<li>Cluster method assessment based on OTU inflation.
<ul>
<li>Clustering methods have also been evaluated based on the number of predicted OTUs relative to the number of expected OTUs <span class="citation">(Huse et al. <a href="#ref-Huse2010">2010</a>, <span class="citation">Kopylova et al. (<a href="#ref-Kopylova2014">2014</a>)</span>)</span>.<br />
</li>
<li>Here either simulated or data from a mock community, where the number of OTUs is based on the number of strains or genomes used to generate the assessment dataset.<br />
</li>
<li>The limitation to this approach is that it does not account for contaminants when <strong>in vitro</strong> mock communities are used, and neither <strong>in silico</strong> or <strong>in vitro</strong> datasets represent the true complexity and diversity of environmental samples.</li>
</ul></li>
</ul>
</div>
</div>
</div>
<div id="downstream-analysis-1" class="section level2">
<h2><span class="header-section-number">5.3</span> Downstream analysis</h2>
<div id="taxonomic-classification" class="section level3">
<h3><span class="header-section-number">5.3.1</span> taxonomic classification</h3>
</div>
<div id="differential-abundance-1" class="section level3">
<h3><span class="header-section-number">5.3.2</span> differential abundance</h3>
<ul>
<li>Key parameters
<ul>
<li>sample size<br />
</li>
<li>effect size (fold change)<br />
</li>
<li>abundance (count)<br />
</li>
</ul></li>
<li><span class="citation">Jonsson et al. (<a href="#ref-Jonsson2016">2016</a>)</span> evaluation of different methods for whole metagenome sequencing
<ul>
<li>Compared ROC for simulated datasets<br />
</li>
<li>sampled datasets and provided truth set for differentially abundant genes by replacing count values using a binomial distribution with <span class="math inline">\(1/q\)</span> where <span class="math inline">\(q\)</span> is defined as the effect size.</li>
</ul></li>
</ul>
</div>
<div id="ecological-diversity" class="section level3">
<h3><span class="header-section-number">5.3.3</span> ecological diversity</h3>

</div>
</div>
</div>
<h3> References</h3>
<div id="refs" class="references">
<div id="ref-Brooks2015">
<p>Brooks, J Paul, David J Edwards, Michael D Harwich, Maria C Rivera, Jennifer M Fettweis, Myrna G Serrano, Robert A Reris, et al. 2015. “The truth about metagenomics: quantifying and counteracting bias in 16S rRNA studies.” <em>BMC Microbiology</em> 15 (1): 66. doi:<a href="https://doi.org/10.1186/s12866-015-0351-6">10.1186/s12866-015-0351-6</a>.</p>
</div>
<div id="ref-Kunin2010">
<p>Kunin, Victor, Anna Engelbrektson, Howard Ochman, and Philip Hugenholtz. 2010. “Wrinkles in the rare biosphere: pyrosequencing errors can lead to artificial inflation of diversity estimates.” <em>Environmental Microbiology</em> 12 (1): 118–23. doi:<a href="https://doi.org/10.1111/j.1462-2920.2009.02051.x">10.1111/j.1462-2920.2009.02051.x</a>.</p>
</div>
<div id="ref-Schirmer2015">
<p>Schirmer, Melanie, Umer Z. Ijaz, Rosalinda D’Amore, Neil Hall, William T. Sloan, and Christopher Quince. 2015. “Insight into biases and sequencing errors for amplicon sequencing with the Illumina MiSeq platform.” <em>Nucleic Acids Research</em> 43 (6). doi:<a href="https://doi.org/10.1093/nar/gku1341">10.1093/nar/gku1341</a>.</p>
</div>
<div id="ref-Westcott2015">
<p>Westcott, Sarah L, and Patrick D Schloss. 2015. “De novo clustering methods out-perform reference-based methods for assigning 16S rRNA gene sequences to operational taxonomic units.” <em>PeerJ</em> 3: e1487. doi:<a href="https://doi.org/10.7287/peerj.preprints.1466v1">10.7287/peerj.preprints.1466v1</a>.</p>
</div>
<div id="ref-Kopylova2014">
<p>Kopylova, Evguenia, Jose A Navas-molina, Céline Mercier, and Zech Xu. 2014. “Open-Source Sequence Clustering Methods Improve the State Of the Art.” <em>MSystems</em> 1 (1): 1–16. doi:<a href="https://doi.org/10.1128/mSystems.00003-15.Editor">10.1128/mSystems.00003-15.Editor</a>.</p>
</div>
<div id="ref-Schloss2016">
<p>Schloss, Patrick D. 2016. “Application of database-independent approach to assess the quality of OTU picking methods.”</p>
</div>
<div id="ref-Schloss2011">
<p>Schloss, Patrick D., and Sarah L. Westcott. 2011. “Assessing and Improving Methods Used in Operational Taxonomic Unit-Based Approaches for 16S rRNA Gene Sequence Analysis.” <em>Applied and Environmental Microbiology</em> 77 (10): 3219–26. doi:<a href="https://doi.org/10.1128/AEM.02810-10">10.1128/AEM.02810-10</a>.</p>
</div>
<div id="ref-Ghodsi2011">
<p>Ghodsi, Mohammadreza, Bo Liu, and Mihai Pop. 2011. “DNACLUST: accurate and efficient clustering of phylogenetic marker genes.” <em>BMC Bioinformatics</em> 12 (1). BioMed Central Ltd: 271. doi:<a href="https://doi.org/10.1186/1471-2105-12-271">10.1186/1471-2105-12-271</a>.</p>
</div>
<div id="ref-He2015">
<p>He, Yan, J Gregory Caporaso, Xiao-Tao Jiang, Hua-Fang Sheng, Susan M Huse, Jai Ram Rideout, Robert C Edgar, et al. 2015. “Stability of operational taxonomic units: an important but neglected property for analyzing microbial diversity.” <em>Microbiome</em> 3 (1). ??? 20. doi:<a href="https://doi.org/10.1186/s40168-015-0081-x">10.1186/s40168-015-0081-x</a>.</p>
</div>
<div id="ref-Chen2013">
<p>Chen, Wei, Clarence K. Zhang, Yongmei Cheng, Shaowu Zhang, and Hongyu Zhao. 2013. “A Comparison of Methods for Clustering 16S rRNA Sequences into OTUs.” <em>PLoS ONE</em> 8 (8). doi:<a href="https://doi.org/10.1371/journal.pone.0070837">10.1371/journal.pone.0070837</a>.</p>
</div>
<div id="ref-Schmidt2015">
<p>Schmidt, Thomas S B, Jo??o F. Matias Rodrigues, and Christian von Mering. 2015. “Limits to robustness and reproducibility in the demarcation of operational taxonomic units.” <em>Environmental Microbiology</em> 17 (5): 1689–1706. doi:<a href="https://doi.org/10.1111/1462-2920.12610">10.1111/1462-2920.12610</a>.</p>
</div>
<div id="ref-Huse2010">
<p>Huse, Susan M, David Mark Welch, Hilary G Morrison, and Mitchell L Sogin. 2010. “Ironing out the wrinkles in the rare biosphere through improved OTU clustering.” <em>Environmental Microbiology</em> 12 (7): 1889–98. doi:<a href="https://doi.org/10.1111/j.1462-2920.2010.02193.x">10.1111/j.1462-2920.2010.02193.x</a>.</p>
</div>
<div id="ref-Jonsson2016">
<p>Jonsson, Viktor, Tobias Österlund, Olle Nerman, and Erik Kristiansson. 2016. “Statistical evaluation of methods for identification of differentially abundant genes in comparative metagenomics.” <em>BMC Genomics</em> 17 (1). BioMed Central: 78. doi:<a href="https://doi.org/10.1186/s12864-016-2386-y">10.1186/s12864-016-2386-y</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="measurement-process-1.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="mixture-study.html" class="navigation navigation-next " aria-label="Next page""><i class="fa fa-angle-right"></i></a>

<script src="libs/gitbook/js/app.min.js"></script>
<script src="libs/gitbook/js/lunr.js"></script>
<script src="libs/gitbook/js/plugin-search.js"></script>
<script src="libs/gitbook/js/plugin-sharing.js"></script>
<script src="libs/gitbook/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook/js/plugin-bookdown.js"></script>
<script src="libs/gitbook/js/jquery.highlight.js"></script>
<script>
require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/inst/examples/30-Method-Assessment.Rmd",
"text": "Edit"
},
"download": null,
"toc": {
"collapse": false
},
"toolbar": {
"position": "fixed"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
